Publications
================
Marc Vaudel
2019-08-01

This repository lists the scientific publications I co-authored as of 1
August 2019. Citation metrics as obtained from the NIH
[iCite](icite.od.nih.gov) are available
[here](docs/icite/icite_report_01.08.2019.xlsx).

<br>

> <sup>\*</sup> shared first, <sup>:email:</sup> corresponding
> authorship, <sup>†</sup> shared last.

# Five highlighted original articles

1.  Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>, *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019. \[[Preprint](https://doi.org/10.1101/375097)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

<table>

<tr>

<td width="150">

![PathwayMatcher](docs/figures/PathwayMatcher.png?raw=true
"PathwayMatcher")

</td>

<td>

*<sub>PathwayMatcher can build biological networks of different levels
of granularity using the [Reactome](reactome.org) pathway database, and
map different types of omics data to it. Notably, it can build a network
to the level of
[proteoforms](https://www.nature.com/articles/nmeth.2369), where genes
and proteins are separated according to their post-translational state.
Hence, interactions are more specific, and interactions between
different forms of the same proteins become visible as connections
between different nodes, thus providing a new dimension to study
biological networks.</sub>* <br>

</td>

</tr>

</table>

<br>

2.  Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>, *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications,
    *Accepted*. \[[Preprint](https://doi.org/10.1101/478255)\]
    \[[doi: 10.1101/478255](https://doi.org/10.1101/478255)\] \[[pdf
    (preprint)](https://www.biorxiv.org/content/biorxiv/early/2018/11/25/478255.full.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![MH time lapse](docs/figures/mh_time-lapse.gif?raw=true
"MH time lapse")

</td>

<td>

*<sub>The association between genetic variation and phenotypes, *e.g.*
with the Body Mass Index (BMI), is often presented as static, yet it
changes over the course of life. Here we present genome-wide association
results during the first years of life, from birth to age 8, making
possibly one of the first genetic time lapse. It represents 12 GWASs on
the [MoBa
cohort](https://www.fhi.no/studier/moba/forskere/sporreskjemaer---mor-og-barn-unders/),
for which we bust a complete bioinformatic environment from
scratch.</sub>* <br>

</td>

</tr>

</table>

<br>

3.  Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>, *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018.
    \[[Preprint](https://doi.org/10.1101/333492)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://doi.org/10.1021/acs.jproteome.8b00464)\]
    \[[pdf
    (preprint)](https://www.biorxiv.org/content/biorxiv/early/2018/05/30/333492.full.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![Pathways](docs/figures/TOC_bram.png?raw=true "Pathways")

</td>

<td>

*<sub>An analysis of the knowledge consolidated in the
[Reactome](reactome.org) pathway database. How are biochemical pathways
modelled? What is the current extent of our knowledge? How does this
affect the interpretation of biomedical (proteomics) data?</sub>* <br>

</td>

</tr>

</table>

<br>

4.  Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>, *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017.
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

<table>

<tr>

<td width="150">

![PeptideMapper](docs/figures/PeptideMapper.png?raw=true
"PeptideMapper")

</td>

<td>

*<sub>PeptideMapper is an efficient sequence alignment tool for
proteomics. Sequences can be partial with missing sections represented
as mass gaps or wild cards. It accounts for post-translational
modifications and sequence variants during mapping.</sub>* <br>

</td>

</tr>

</table>

<br>

5.  **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H, *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015.
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (post-peer-review, pre-copyedit
    version)](docs/aam/peptideshaker_submitted.pdf)\] \[[Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![PeptideShaker](docs/figures/SearchGUI_and_PS-overview-figure.png?raw=true
"PeptideShaker")

</td>

<td>

*<sub>PeptideShaker is a bioinformatic tools that processes proteomics
identification results. It has both rich graphical and command line
interfaces and can operate on in-house data just like on public
proteomics data from the [EMBL-EBI
PRIDE](https://www.ebi.ac.uk/pride/archive/) repository. It can handle
results from database searches as well as *de novo* sequencing, with
support for sequence variants.</sub>* <br>

</td>

</tr>

</table>

<br>

# Original Articles

(Work in progress)

38. Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>, *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019. \[[bioRxiv](https://doi.org/10.1101/375097)\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

39. Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>, *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications,
    *Accepted*. \[[bioRxiv](https://doi.org/10.1101/478255)\] \[PubMed\]
    \[[doi: 10.1101/478255](https://doi.org/10.1101/478255)\] \[[pdf
    (preprint)](https://www.biorxiv.org/content/biorxiv/early/2018/11/25/478255.full.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

40. Bradfield JP<sup>\(</sup>, Vogelezang S<sup>\)</sup>, …, **Vaudel
    M**, …, Jaddoe VWV<sup>†</sup>, Grant SFA<sup>†,:email:</sup>, *A
    Trans-ancestral Meta-Analysis of Genome-Wide Association Studies
    Reveals Loci Associated with Childhood Obesity*, Human Molecular
    Genetics, 2019

41. Warrington MW<sup>\(</sup>, Beaumont R<sup>\)</sup>, Horikoshi
    M<sup>\(</sup>, Day FR<sup>\)</sup>, Helgeland Ø<sup>$</sup>, …,
    **Vaudel M**, …, Njølstad PR, Johansson S, Ong KK<sup>†</sup>,
    McCarthy MI<sup>†</sup>, Perry JRB<sup>†</sup>, Evans
    DM<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>, *Maternal
    and fetal genetic effects on birth weight and their relevance to
    cardio-metabolic risk factors*, Nature Genetics, 2019
    (biorxiv.org/content/10.1101/442756v1)

42. Zhang-James Y, **Vaudel M**, Mjaavatten O, Berven FS, Haavik J,
    Faraone SV, *Effect of disease-associated SLC9A9 mutations on
    protein-protein interaction networks: implications for molecular
    mechanisms for ADHD and autism*, ADHD Attention Deficit and
    Hyperactivity Disorders, 2019.

43. Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>, *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018.
    \[[bioRxiv](https://doi.org/10.1101/333492)\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30251541)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00464)\]
    \[[pdf
    (preprint)](https://www.biorxiv.org/content/biorxiv/early/2018/05/30/333492.full.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

44. Barsnes H, **Vaudel M**<sup>:email:</sup>, *SearchGUI: A Highly
    Adaptable Common Interface for Proteomics Search and de Novo
    Engines*, Journal of Proteome Research, 2018.

45. Li K, **Vaudel M**, Zhang B, Ren Y, Wen B<sup>:email:</sup>, *PDV:
    an integrative proteomics data viewer*, Bioinformatics, 2018.

46. Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>, *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017. \[bioRxiv\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

47. Berland S, Toft Bertelsen TL, Aukrust I, Byška J, **Vaudel M**,
    MacAulay N, Houge G<sup>:email:</sup>, *A de novo Ser111Thr in
    aquaporin-4 in a patient with transient brain edema, transient heart
    failure, intellectual disability and slowly progressive brain
    dysfunction*, Molecular Case Studies, 2017.

48. Beaumont RN<sup>\(</sup>, Warrington NM<sup>\)</sup>, …, **Vaudel
    M**, …, Evans DM<sup>†</sup>, Lawlor DA<sup>†,:email:</sup>,
    Feenstra B<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>,
    *Genome-wide association study of offspring birth weight in 86,577
    women highlights maternal genetic effects that are independent of
    fetal genetics*, Human Molecular Genetics, 2017.

49. Chambers MC, Jagtap PD, Johnson JE, McGowan T, Kumar P, Onsongo G,
    Guerrero CR, Barnses H, **Vaudel M**, Martens L, Grüning BA, Cooke
    IR, Heydarian M, Reddy KL, Griffin TJ<sup>:email:</sup>, *An
    accessible proteogenomics informatics resource for cancer
    researchers*, Cancer Research, 2017.

50. Vizcaíno JA, Meyer G, Perkins SR, Barsnes H, **Vaudel M**,
    Perez-Riverol Y, Terment T, Uszkoreit J, Eineacher M, Fischer L,
    Rappsilber J, Netz E, Walzer M, Kohlbacher O, Leitner A, Chalkley
    RJ, Ghali F, Martínez-Bartolomé S, Deutsch EW, Johnes
    AR<sup>:email:</sup>, *The mzIdentML data standard version 1.2,
    supporting advances in proteome informatics*, Mol Cell Proteomics,
    2017.

51. Leprevost FdV, Aflitos SA, Grüning BA, Röst HL, Uszkoreit J, Barsnes
    H, **Vaudel M**, Moreno P, Gatto L, Weber J, Bai M, Jimenez RC,
    Sachsenberg T, Pfeufferz J, Alvarez RV, Griss J, Nesvizhskii AI,
    Perez-Riverol Y<sup>:email:</sup>, *BioContainers: An open-source
    and community-driven framework for software standardization*,
    Bioinformatics, 2017.

52. Kroksveen AC<sup>:email:</sup>, Guldbrandsen A, **Vaudel M**, Lereim
    RR, Barsnes H, Myr KM, Torkildsen Ø, Berven FS<sup>:email:</sup>,
    *In-Depth Cerebrospinal Fluid Quantitative Proteome and
    Deglycoproteome Analysis: Presenting a Comprehensive Picture of
    Pathways and Processes Affected by Multiple Sclerosis*, Journal of
    Proteome Research, 2017.

53. Hernandez-Valladares M<sup>\(,:email:</sup>, Aasebø E<sup>\)</sup>,
    Mjaavatten O, **Vaudel M**, Bruserud Ø, Berven FS, Selheim F,
    *Reliable FASP-based procedures for optimal quantitative proteomic
    and phosphoproteomic analysis on samples from acute myeloid leukemia
    patients*, Biol Proced Online, 2016.

54. Aasebø
    E<sup>\(</sup>, Mjaavatten O, **Vaudel M**, Farag Y, Selheim F, Berven FS, Bruserud Ø, Hernandez-Valladares M<sup>\),:email:</sup>,
    *Freezing effects on the acute myeloid leukemia cell proteome and
    phosphoproteome revealed using optimal quantitative workflows*,
    Journal of Proteomics, 2016.

55. Opsahl JA, **Vaudel M**, Guldbrandsen A, Aasebø E, Van Pesch V,
    Franciotta D, Myhr KM, Barsnes H, Berle M, Torkildsen Ø, Kroksveen
    AC, Berven FS<sup>:email:</sup>, *Label free analysis of human
    cerebrospinal fluid addressing various normalization strategies and
    revealing protein groups affected by multiple sclerosis*,
    Proteomics, 2016.

56. **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H, *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015. \[bioRxiv\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (post-peer-review, pre-copyedit
    version)](docs/aam/peptideshaker_submitted.pdf)\] \[[Request a
    reprint](mailto:marc.vaudel@uib.no)\]

57. Verheggen K, Maddelein D, Hulstaert N, Martens L<sup>:email:</sup>,
    Barsnes H, Vaudel M, *Pladipus enables universal distributed
    computing in proteomics bioinformatics*, Journal of Proteome
    Research, 2015.

58. Barsnes H<sup>:email:</sup>, **Vaudel M**, Martens L, *JSparklines:
    Making tabular proteomics data come alive*, Proteomics, 2015.

59. Bjørlykke Y, Vethe H, **Vaudel M**, Barsnes H, Berven FS, Tjora E,
    Raeder H<sup>:email:</sup>, *Carboxyl-Ester Lipase Maturity-Onset
    Diabetes of the Young Disease Protein Biomarkers in
    Secretin-Stimulated Duodenal Juice*, Journal of Proteome Research,
    2015.

60. Muth T<sup>\(</sup>, Weilnböck L<sup>\)</sup>, Rapp E, Huber CG,
    Martens L, Vaudel M<sup>:email:</sup>, Barsnes H, *DeNovoGUI: an
    open source graphical user interface for de novo sequencing of
    tandem mass spectra*, Journal of Proteome Research, 2014.

61. Helgeland E, Breivik LE, **Vaudel M**, Svendsen OS, Garberg H,
    Nordrehaug JE, Berven FS, Jonassen AK<sup>:email:</sup>, *Exploring
    the human plasma proteome for humoral mediators of remote ischemic
    preconditioning - a word of caution*, PLoS One, 2014.

62. Vethe H, Finne K, Skogstrand T, **Vaudel M**, Vikse BE, Hultström M,
    Placier S, Scherer A, Tenstad O, Marti HP<sup>:email:</sup>,
    *Distinct protein signature of hypertension-induced damage in the
    renal proteome of the two-kidney, one-clip rat model*, Journal of
    Hypertension, 2015.

63. Aasebø E, **Vaudel M**, Mjaavatten O, Gaudsdal G, Van der Burgh A,
    Gjertsen BT, Døskeland SO, Brusserud O, Berven FS, Selheim
    F<sup>:email:</sup>, *Performance of super-SILAC based quantitative
    proteomics for comparison of different acute myeloid leukemia (AML)
    cell lines*, Proteomics, 2014.

64. **Vaudel M**, Venne S, Berven FS, Zahedi RP, Martens L, Barsnes
    H<sup>:email:</sup>, *Shedding light on black boxes in protein
    identification*, Proteomics, 2014.

65. Beck F, Geiger J, Gambaryan S, Veit J, **Vaudel M**, Nollau P,
    Kohlbacher O, Martens L, Walter U, Sickmann A<sup>:email:</sup>,
    Zahedi RP<sup>:email:</sup>, *Time-resolved characterization of
    cAMP/PKA-dependent signaling reveals that platelet inhibition is a
    concerted process involving multiple signaling pathways*, Blood,
    2014.

66. **Vaudel M**, Breiter D, Beck F, Rahnenführer J, Martens
    L<sup>:email:</sup>, Zahedi RP, *D-score: a search engine
    independent MD-score*, Proteomics, 2013.

67. Burkhart JM<sup>\(</sup>, Vaudel M<sup>\)</sup>, Gambaryan S, Radau
    S, Walter U, Martens L, Geiger J<sup>:email:</sup>, Sickmann A, and
    Zahedi RP<sup>:email:</sup>, *The first comprehensive and
    quantitative analysis of human platelet protein composition allows
    the comparative analysis of structural and functional pathways in
    human platelets*, Blood, 2012.

68. **Vaudel M**<sup>\(</sup>, Burkhart JM<sup>\)</sup>, Radau S, Zahedi
    RP, Martens L<sup>:email:</sup>, Sickmann A, *Integral
    Quantification Accuracy estimation for Reporter Ion based
    quantitative proteomics (iQuARI)*, Journal of Proteome Research,
    2012.

69. **Vaudel M**<sup>\(</sup>, Burkhart JM<sup>\)</sup>, Breiter D,
    Zahedi RP, Sickmann A<sup>:email:</sup>, Martens L, *A complex
    standard for protein identification, designed by evolution*, Journal
    of Proteome Research, 2012.

70. Vaudel M<sup>\(</sup>, Barsnes H<sup>\),:email:</sup>, Berven FS,
    Sickmann A, Martens L, *SearchGUI: an open-source graphical user
    interface for simultaneous OMSSA and X\!Tandem searches*,
    Proteomics, 2011.

71. Colaert N, Barsnes H, **Vaudel M**, Helsens K, Timmerman E, Sickmann
    A, Gevaert K, Martens L<sup>:email:</sup>, *Thermo-msf-parser: an
    open source Java library to parse and visualize Thermo Proteome
    Discoverer msf files*, Journal of Proteome Research, 2011.

72. **Vaudel M**<sup>\(</sup>, Burkhart JM<sup>\)</sup>, Sickmann A,
    Martens L<sup>:email:</sup>, Zahedi RP, *Peptide identification
    quality control*, Proteomics, 2011.

73. Barsnes H, **Vaudel M**, Colaert N, Helsens K, Sickmann A, Berven
    FS, Martens L<sup>:email:</sup>, *compomics-utilities: an
    open-source Java library for computational proteomics*, BMC
    Bioinformatics, 2011.

74. Burkhart JM<sup>\(</sup>, **Vaudel M**<sup>\)</sup>, Zahedi RP,
    Martens L, Sickmann A<sup>:email:</sup>, *iTRAQ protein
    quantification: a quality-controlled workflow*, Proteomics, 2011.

75. Muth T, **Vaudel M**, Barsnes H, Martens L<sup>:email:</sup>,
    Sickmann A, *XTandem Parser: An open-source library to parse and
    analyse X\!Tandem MS/MS search results*, Proteomics, 2010.

<br>

# Reviews

(Coming soon\!) <br>

# Book Chapters

(Coming soon\!) <br>

# Outreach - Proceeding

(Coming soon\!) <br>

# PhD thesis

*Detailed analysis of the blood platelet proteome through the use of
innovative bioinformatics methods for protein identification,
quantification and post-translational modification localization.*
[link](hdl.handle.net/1854/LU-3076565) <br>

# About

This document was generated from this [Rmd](README.Rmd). Links to pdfs
are obtained from [Unpaywall](unpaywall.org), queried using the
[roadoi](https://cran.r-project.org/web/packages/roadoi/vignettes/intro.html)
package.

For personal use only, no copyright infringement intended. <br>

## R Session Info

``` r
sessionInfo()
```

    ## R version 3.6.1 (2019-07-05)
    ## Platform: x86_64-w64-mingw32/x64 (64-bit)
    ## Running under: Windows 10 x64 (build 17134)
    ## 
    ## Matrix products: default
    ## 
    ## locale:
    ## [1] LC_COLLATE=Norwegian Bokmål_Norway.1252 
    ## [2] LC_CTYPE=Norwegian Bokmål_Norway.1252   
    ## [3] LC_MONETARY=Norwegian Bokmål_Norway.1252
    ## [4] LC_NUMERIC=C                            
    ## [5] LC_TIME=Norwegian Bokmål_Norway.1252    
    ## 
    ## attached base packages:
    ## [1] stats     graphics  grDevices utils     datasets  methods   base     
    ## 
    ## other attached packages:
    ## [1] roadoi_0.5.2
    ## 
    ## loaded via a namespace (and not attached):
    ##  [1] Rcpp_1.0.2       knitr_1.23       magrittr_1.5     tidyselect_0.2.5
    ##  [5] xtable_1.8-4     R6_2.4.0         rlang_0.4.0      plyr_1.8.4      
    ##  [9] stringr_1.4.0    httr_1.4.0       dplyr_0.8.3      tools_3.6.1     
    ## [13] xfun_0.8         miniUI_0.1.1.1   htmltools_0.3.6  yaml_2.2.0      
    ## [17] assertthat_0.2.1 digest_0.6.20    tibble_2.1.3     crayon_1.3.4    
    ## [21] shiny_1.3.2      tidyr_0.8.3      later_0.8.0      purrr_0.3.2     
    ## [25] promises_1.0.1   curl_4.0         mime_0.7         glue_1.3.1      
    ## [29] evaluate_0.14    rmarkdown_1.14   stringi_1.4.3    compiler_3.6.1  
    ## [33] pillar_1.4.2     jsonlite_1.6     httpuv_1.5.1     pkgconfig_2.0.2
