Publications
================
Marc Vaudel
2019-08-01

This repository lists the scientific publications I co-authored as of 1
August 2019. Citation metrics as obtained from the NIH
[iCite](icite.od.nih.gov) are available
[here](docs/icite/icite_report_01.08.2019.xlsx).

> <sup>\*</sup> shared first, <sup>:email:</sup> corresponding
> authorship, <sup>†</sup> shared last.

<br>

# Five highlighted original articles

1.  Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>, *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019. \[[bioRxiv](https://doi.org/10.1101/375097)\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf
    (Unpaywall)](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

<table>

<tr>

<td width="150">

![PathwayMatcher](docs/figures/PathwayMatcher.png?raw=true
"PathwayMatcher")

</td>

<td>

*<sub>PathwayMatcher can build biological networks of different levels
of granularity using the [Reactome](reactome.org) pathway database, and
map different types of omics data to it. Notably, it can build a network
to the level of
[proteoforms](https://www.nature.com/articles/nmeth.2369), where genes
and proteins are separated according to their post-translational state.
Hence, interactions are more specific, and interactions between
different forms of the same proteins become visible as connections
between different nodes, thus providing a new dimension to study
biological networks.</sub>* <br>

</td>

</tr>

</table>

<br>

2.  Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>, *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications,
    *Accepted*. \[[bioRxiv](https://doi.org/10.1101/478255)\] \[PubMed\]
    \[[doi: 10.1101/478255](https://doi.org/10.1101/478255)\] \[[pdf
    (Unpaywall)](https://www.biorxiv.org/content/biorxiv/early/2018/11/25/478255.full.pdf)\]

<table>

<tr>

<td width="150">

![MH time lapse](docs/figures/mh_time-lapse.gif?raw=true
"MH time lapse")

</td>

<td>

*<sub>The association between genetic variation and phenotypes, *e.g.*
with the Body Mass Index (BMI), is often presented as static, yet it
changes over the course of life. Here we present genome-wide association
results during the first years of life, from birth to age 8, making
possibly one of the first genetic time lapse. It represents 12 GWASs on
the [MoBa
cohort](https://www.fhi.no/studier/moba/forskere/sporreskjemaer---mor-og-barn-unders/),
for which we bust a complete bioinformatic environment from
scratch.</sub>* <br>

</td>

</tr>

</table>

<br>

3.  Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>, *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018.
    \[[bioRxiv](https://doi.org/10.1101/333492)\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30251541)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00464)\]
    \[[pdf
    (Unpaywall)](https://www.biorxiv.org/content/biorxiv/early/2018/05/30/333492.full.pdf)\]

<table>

<tr>

<td width="150">

![Pathways](docs/figures/TOC_bram.png?raw=true "Pathways")

</td>

<td>

*<sub>An analysis of the knowledge consolidated in the
[Reactome](reactome.org) pathway database. How are biochemical pathways
modelled? What is the current extent of our knowledge? How does this
affect the interpretation of biomedical (proteomics) data?</sub>* <br>

</td>

</tr>

</table>

<br>

4.  Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>, *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017. \[bioRxiv\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf
    (Unpaywall)](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

<table>

<tr>

<td width="150">

![PeptideMapper](docs/figures/PeptideMapper.png?raw=true
"PeptideMapper")

</td>

<td>

*<sub>PeptideMapper is an efficient sequence alignment tool for
proteomics. Sequences can be partial with missing sections represented
as mass gaps or wild cards. It accounts for post-translational
modifications and sequence variants during mapping.</sub>* <br>

</td>

</tr>

</table>

<br>

5.  **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H, *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015. \[bioRxiv\]
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[No
    Open Access found [Request a
reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![PeptideShaker](docs/figures/SearchGUI_and_PS-overview-figure.png?raw=true
"PeptideShaker")

</td>

<td>

*<sub>PeptideShaker is a bioinformatic tools that processes proteomics
identification results. It has both rich graphical and command line
interfaces and can operate on in-house data just like on public
proteomics data from the [EMBL-EBI
PRIDE](https://www.ebi.ac.uk/pride/archive/) repository. It can handle
results from database searches as well as *de novo* sequencing, with
support for sequence variants.</sub>* <br>

</td>

</tr>

</table>

<br>

# Original Articles

(Coming soon\!)

# About

This document was generated from this [Rmd](README.Rmd).
[Unpaywall](unpaywall.org) was queried using the
[roadoi](https://cran.r-project.org/web/packages/roadoi/vignettes/intro.html)
package.

For personal use only, no copyright infringement intended.

## R Session Info

``` r
sessionInfo()
```

    ## R version 3.6.1 (2019-07-05)
    ## Platform: x86_64-w64-mingw32/x64 (64-bit)
    ## Running under: Windows 10 x64 (build 17134)
    ## 
    ## Matrix products: default
    ## 
    ## locale:
    ## [1] LC_COLLATE=Norwegian Bokmål_Norway.1252 
    ## [2] LC_CTYPE=Norwegian Bokmål_Norway.1252   
    ## [3] LC_MONETARY=Norwegian Bokmål_Norway.1252
    ## [4] LC_NUMERIC=C                            
    ## [5] LC_TIME=Norwegian Bokmål_Norway.1252    
    ## 
    ## attached base packages:
    ## [1] stats     graphics  grDevices utils     datasets  methods   base     
    ## 
    ## other attached packages:
    ## [1] roadoi_0.5.2
    ## 
    ## loaded via a namespace (and not attached):
    ##  [1] Rcpp_1.0.2       knitr_1.23       magrittr_1.5     tidyselect_0.2.5
    ##  [5] xtable_1.8-4     R6_2.4.0         rlang_0.4.0      plyr_1.8.4      
    ##  [9] stringr_1.4.0    httr_1.4.0       dplyr_0.8.3      tools_3.6.1     
    ## [13] xfun_0.8         miniUI_0.1.1.1   htmltools_0.3.6  yaml_2.2.0      
    ## [17] assertthat_0.2.1 digest_0.6.20    tibble_2.1.3     crayon_1.3.4    
    ## [21] shiny_1.3.2      tidyr_0.8.3      later_0.8.0      purrr_0.3.2     
    ## [25] promises_1.0.1   curl_4.0         mime_0.7         glue_1.3.1      
    ## [29] evaluate_0.14    rmarkdown_1.14   stringi_1.4.3    compiler_3.6.1  
    ## [33] pillar_1.4.2     jsonlite_1.6     httpuv_1.5.1     pkgconfig_2.0.2
