---
title: "Publications"
author: Marc Vaudel
date: 2019-08-01
output: 
    github_document:
    toc: true
    toc_depth: 1
---


This repository lists the scientific publications I co-authored as of 1 August 2019. Citation metrics as obtained from the NIH [iCite](icite.od.nih.gov) are available [here](docs/icite/icite_report_01.08.2019.xlsx). For personal use only, no copyright infringement intended.

> <sup>*</sup> shared first, <sup>:email:</sup> corresponding authorship, <sup>†</sup> shared last. 

<br>

# Five highlighted original articles

1. Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S, Njølstad PR, Barsnes H, Hermjakob H, **Vaudel M**<sup>:email:</sup>, _PathwayMatcher: proteoform-centric network construction enables fine-granularity multi-omics pathway mapping_, GigaScience, 2019. [[bioRxiv](https://doi.org/10.1101/375097)] [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31363752)] [[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)] [[pdf](docs/pdf/giz088.pdf)]</font>

<table>
<tr>
<td width="150">
![PathwayMatcher](docs/figures/PathwayMatcher.png?raw=true "PathwayMatcher")
</td>
<td>
_<sub>PathwayMatcher can build biological networks of different levels of granularity using the [Reactome](reactome.org) pathway database, and map different types of omics data to it. Notably, it can build a network to the level of [proteoforms](https://www.nature.com/articles/nmeth.2369), where genes and proteins are separated according to their post-translational state. Hence, interactions are more specific, and interactions between different forms of the same proteins become visible as connections between different nodes, thus providing a new dimension to study biological networks.</sub>_
<br>
</td>
</tr>
</table>
<br>


2. Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>, _Genome-wide association study reveals a dynamic role of common genetic variation in infant and early childhood growth_, Nature Communications, _Accepted_. [[bioRxiv](https://doi.org/10.1101/478255)] [PubMed] [[doi: 10.1101/478255](https://doi.org/10.1101/478255)] [[pdf](docs/pdf/478255.full.pdf)]

<table>
<tr>
<td width="150">
![MH time lapse](docs/figures/mh_time-lapse.gif?raw=true "MH time lapse")
</td>
<td> 
_<sub>The association between genetic variation and phenotypes, _e.g._ with the Body Mass Index (BMI), is often presented as static, yet it changes over the course of life. Here we present genome-wide association results during the first years of life, from birth to age 8, making possibly one of the first genetic time lapse. It represents 12 GWASs on the [MoBa cohort](https://www.fhi.no/studier/moba/forskere/sporreskjemaer---mor-og-barn-unders/), for which we bust a complete bioinformatic environment from scratch.</sub>_
<br>
</td>
</tr>
</table>
<br>


3. Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel M**<sup>:email:</sup>, _Analyzing the Structure of Pathways and Its Influence on the Interpretation of Biomedical Proteomics Data Sets_, Journal of Proteome Research, 2018. [[bioRxiv](https://doi.org/10.1101/333492)] [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30251541)] [[doi: 10.1021/acs.jproteome.8b00464](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00464)] [[pdf](docs/pdf/acs.jproteome.8b00464.pdf)]

<table>
<tr>
<td width="150">
![Pathways](docs/figures/TOC_bram.png?raw=true "Pathways")
</td>
<td>
_<sub>An analysis of the knowledge consolidated in the [Reactome](reactome.org) pathway database. How are biochemical pathways modelled? What is the current extent of our knowledge? How does this affect the interpretation of biomedical (proteomics) data?</sub>_
<br>
</td>
</tr>
</table>
<br>


4. Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>, _PeptideMapper: Efficient and Versatile Amino Acid Sequence and Tag Mapping_, Bioinformatics, 2017. [bioRxiv] [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28334306)] [[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)] [[pdf](docs/pdf/btx122.pdf)]

<table>
<tr>
<td width="150">
![PeptideMapper](docs/figures/PeptideMapper.png?raw=true "PeptideMapper")
</td>
<td>
_<sub>PeptideMapper is an efficient sequence alignment tool for proteomics. Sequences can be partial with missing sections represented as mass gaps or wild cards. It accounts for post-translational modifications and sequence variants during mapping.</sub>_
<br>
</td>
</tr>
</table>
<br>


5. **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann A, Martens L<sup>:email:</sup>, Barsnes H, _PeptideShaker enables reanalysis of mass spectrometry-derived proteomics datasets_, Nature Biotechnology, 2015. [bioRxiv] [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25574629)] [[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)] [[pdf](docs/pdf/nbt.3109.pdf)]

<table>
<tr>
<td width="150">
![PeptideMapper](docs/figures/PeptideMapper.png?raw=true "PeptideMapper")
</td>
<td>
_<sub>PeptideMapper is an efficient sequence alignment tool for proteomics. Sequences can be partial with missing sections represented as mass gaps or wild cards. It accounts for post-translational modifications and sequence variants during mapping.</sub>_
<br>
</td>
</tr>
</table>
<br>

# Original Articles


