Publications
================
Marc Vaudel
2019-08-01

[![DOI](https://zenodo.org/badge/200110790.svg)](https://zenodo.org/badge/latestdoi/200110790)

This repository lists the scientific publications co-authored by [Marc
Vaudel](https://www.uib.no/en/persons/Marc.Vaudel) as of 16.03.2021.
Note: For the sake of brevity, some author lists are truncated as
indicated with `...`, apologies if your name went missing.

  - [ORCID: 0000-0003-1179-9578](https://orcid.org/0000-0003-1179-9578)
  - [NIH
    iCite](https://icite.od.nih.gov/analysis?search_id=87lghf7z77q1eloy)
  - [Google
    Scholar](https://scholar.google.com/citations?user=JTK8FTkAAAAJ)

<br>

> <sup>1</sup> shared first, <sup>:email:</sup> correspondence,
> <sup>†</sup> shared last.

<br>

Note: for the sake of brevity, author lists might be trimmed.

## Five Highlighted Original Articles

1.  Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>. *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[Preprint](https://doi.org/10.1101/375097)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

<table>

<tr>

<td width="150">

![PathwayMatcher](docs/figures/PathwayMatcher.png?raw=true
"PathwayMatcher")

</td>

<td>

*<sub>PathwayMatcher can build biological networks of different levels
of granularity using the [Reactome](reactome.org) pathway database, and
map different types of omics data to it. Notably, it can build a network
to the level of
[proteoforms](https://www.nature.com/articles/nmeth.2369), where genes
and proteins are separated according to their post-translational state.
Hence, interactions are more specific, and interactions between
different forms of the same proteins become visible as connections
between different nodes, thus providing a new dimension to study
biological networks.</sub>* <br>

</td>

</tr>

</table>

<br>

2.  Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>. *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31575865)\]
    \[[Preprint](https://doi.org/10.1101/478255)\]
    \[[doi: 10.1038/s41467-019-12308-0](https://doi.org/10.1038/s41467-019-12308-0)\]
    \[[pdf](https://www.nature.com/articles/s41467-019-12308-0.pdf)\]

<table>

<tr>

<td width="150">

![MH time lapse](docs/figures/mh_time-lapse.gif?raw=true
"MH time lapse")

</td>

<td>

*<sub>The association between genetic variation and phenotypes, *e.g.*
with the Body Mass Index (BMI), is often presented as static, yet it
changes over the course of life. Here we present genome-wide association
results during the first years of life, from birth to age 8, making
possibly one of the first genetic time lapse. It represents 12 GWASs on
the [MoBa
cohort](https://www.fhi.no/studier/moba/forskere/sporreskjemaer---mor-og-barn-unders/),
for which we bust a complete bioinformatic environment from
scratch.</sub>* <br>

</td>

</tr>

</table>

<br>

3.  Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30251541)\]
    \[[Preprint](https://doi.org/10.1101/333492)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://doi.org/10.1021/acs.jproteome.8b00464)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/22539/4/Burger%2bet%2bal.%2bAnalysing%2bthe%2bstructure%2bof%2bpathways%2band%2bits%2binfluence%2bon%2bthe%2binterpretation%2bof%2bbiomedical%2bdatasets%2b%2528postprint%2529.pdf)\]

<table>

<tr>

<td width="150">

![Pathways](docs/figures/TOC_bram.png?raw=true "Pathways")

</td>

<td>

*<sub>An analysis of the knowledge consolidated in the
[Reactome](reactome.org) pathway database. How are biochemical pathways
modelled? What is the current extent of our knowledge? How does this
affect the interpretation of biomedical (proteomics) data?</sub>* <br>

</td>

</tr>

</table>

<br>

4.  Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>. *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

<table>

<tr>

<td width="150">

![PeptideMapper](docs/figures/PeptideMapper.png?raw=true
"PeptideMapper")

</td>

<td>

*<sub>PeptideMapper is an efficient sequence alignment tool for
proteomics. Sequences can be partial with missing sections represented
as mass gaps or wild cards. It accounts for post-translational
modifications and sequence variants during mapping.</sub>* <br>

</td>

</tr>

</table>

<br>

5.  **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H. *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (accepted version)](docs/aam/peptideshaker_submitted.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![PeptideShaker](docs/figures/SearchGUI_and_PS-overview-figure.png?raw=true
"PeptideShaker")

</td>

<td>

*<sub>PeptideShaker is a bioinformatic tools that processes proteomics
identification results. It has both rich graphical and command line
interfaces and can operate on in-house data just like on public
proteomics data from the [EMBL-EBI
PRIDE](https://www.ebi.ac.uk/pride/archive/) repository. It can handle
results from database searches as well as *de novo* sequencing, with
support for sequence variants.</sub>* <br>

</td>

</tr>

</table>

<br>

## Original Articles

1.  Eriksson D<sup>1</sup>, Røyrvik EC<sup>1</sup>, Aranda-Guillén
    M<sup>1</sup>, Berger AH, Landegren N, Artaza H, Hallgren Å, Grytaas
    MA, Ström S, Bratland E, Botusan IR, Oftedal BE, Breivik L, **Vaudel
    M**, Helgeland Ø, Falorni A, Jørgensen AP, Hulting A-L, Svartberg J,
    Ekwall O, Fougner KJ, Wahlberg J, Nedrebø BG, Dahlqvist P, The
    Norwegian Addison Registry Study Group, The Swedish Addison Registry
    Study Group, Knappskog PM, Bøe AS, Bensing S, Johansson
    S<sup>†</sup>, Kämpe O<sup>†,:email:</sup>, Husebye
    ES<sup>†,:email:</sup>. *GWAS for autoimmune Addison’s disease
    identifies multiple risk loci and highlights AIRE in disease
    susceptibility*, Nature Communications, 2021.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/33574239)\]
    \[[doi: 10.1038/s41467-021-21015-8](https://doi.org/10.1038/s41467-021-21015-8)\]
    \[[pdf](https://www.nature.com/articles/s41467-021-21015-8.pdf)\]

2.  Sole-Navais P<sup>:email:</sup>, Bacelis J, Helgeland Ø, Modzelewska
    D, **Vaudel M**, Flatley C, Andreassen O, Njølstad PR, Muglia LJ,
    Johansson S, Zhang G, Jacobsson B. *Autozygosity mapping and
    time-to-spontaneous delivery in Norwegian parent-offspring trios*,
    Human Molecular Genetics, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/33291140)\]
    \[[doi: 10.1093/hmg/ddaa255](https://doi.org/10.1093/hmg/ddaa255)\]
    \[[pdf](https://academic.oup.com/hmg/article-pdf/29/23/3845/36194738/ddaa255.pdf)\]

3.  Gupta MK, Vethe H, Softic S, Rao TN, Wagh V, Shirakawa J, Barsnes H,
    **Vaudel M**, Takatani T, Kahraman S, Sakaguchi M, Martinez R, Hu J,
    Bjørlykke Y, Raeder H, Kulkarni RN<sup>:email:</sup>. *Leptin
    Receptor Signaling Regulates Protein Synthesis Pathways and Neuronal
    Differentiation in Pluripotent Stem Cells*, Stem Cell Reports, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/33125875)\]
    \[[doi: 10.1016/j.stemcr.2020.10.001](https://doi.org/10.1016/j.stemcr.2020.10.001)\]
    \[[pdf](http://www.cell.com/article/S2213671120303866/pdf)\]

4.  Vogelezang S<sup>1</sup>, Bradfield JP<sup>1</sup>, …, **Vaudel M**,
    …, Early Growth Genetics Consortium, Jaddoe VWV<sup>†</sup>, Grant
    SFA<sup>†</sup>, Felix JF<sup>†,:email:</sup>. *Novel loci for
    childhood body mass index and shared heritability with adult
    cardiometabolic traits*, PLoS Genetics, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/33045005)\]
    \[[doi: 10.1371/journal.pgen.1008718](https://doi.org/10.1371/journal.pgen.1008718)\]
    \[[pdf](https://doi.org/10.1371/journal.pgen.1008718)\]

5.  Smajlagić D, Lavrichenko K, Berland S, Helgeland Ø, Knudsen GP,
    **Vaudel M**, Haavik J, Knappskog PM, Njølstad PR, Houge G,
    Johansson S<sup>:email:</sup>. *Population prevalence and
    inheritance pattern of recurrent CNVs associated with
    neurodevelopmental disorders in 12,252 newborns and their parents*,
    European Journal of Human Genetics, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/32778765)\]
    \[[doi: 10.1038/s41431-020-00707-7](https://doi.org/10.1038/s41431-020-00707-7)\]
    \[[pdf](https://www.nature.com/articles/s41431-020-00707-7.pdf)\]

6.  Aasebø E, Berven FS, Bartaula-Brevik S, Stokowy T, Hovland R,
    **Vaudel M**, Døskeland SO, McCormack E, Batth TS, Olsen JV,
    Bruserud Ø, Selheim F<sup>†</sup>, Hernandez-Valladares
    M<sup>†,:email:</sup>. *Proteome and Phosphoproteome Changes
    Associated with Prognosis in Acute Myeloid Leukemia*, Cancers, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/32192169)\]
    \[[doi: 10.3390/cancers12030709](https://doi.org/10.3390/cancers12030709)\]
    \[[pdf](https://www.mdpi.com/2072-6694/12/3/709/pdf)\]

7.  Loo LSW, Vethe H, Soetedjo AAP, Paulo JA, Jasmen J, Jackson N,
    Bjørlykke Y, Valdez IA, **Vaudel M**, Barsnes H, Gygi SP, Ræder H,
    Teo AKK<sup>:email:</sup>, Kulkarni RN<sup>:email:</sup>. *Dynamic
    proteome profiling of human pluripotent stem cell-derived pancreatic
    progenitors*, Stem Cells, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31828876)\]
    \[[doi: 10.1002/stem.3135](https://doi.org/10.1002/stem.3135)\]
    \[[pdf](http://europepmc.org/articles/pmc7289403?pdf=render)\]

8.  Bjørlykke Y, Søviknes AM, Hoareau L, Vethe H, Mathisen AF, Chera S,
    **Vaudel M**, Ghila L, Ræder H<sup>:email:</sup>. *Reprogrammed
    cells display distinct proteomic signatures associated with colony
    morphology variability*, Stem Cells International, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31827534)\]
    \[[doi: 10.1155/2019/8036035](https://doi.org/10.1155/2019/8036035)\]
    \[[pdf](https://doi.org/10.1155/2019/8036035)\]

9.  Qiao Z, Zheng J, Helgeland Ø, **Vaudel M**, Johansson S, Njølstad
    PR, Davey Smith G, Warrington NM, Evans D<sup>:email:</sup>.
    *Introducing M-GCTA a software package to estimate maternal (or
    paternal) genetic effects on offspring phenotypes*, Behavior
    Genetics, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31493278)\]
    \[[doi: 10.1007/s10519-019-09969-4](https://doi.org/10.1007/s10519-019-09969-4)\]
    \[[pdf](https://research-information.bris.ac.uk/ws/files/207903720/M_GCTA_R1.pdf)\]

10. Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>. *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[Preprint](https://doi.org/10.1101/375097)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

11. Helgeland Ø, **Vaudel M**, Juliusson P, …, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>. *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31575865)\]
    \[[Preprint](https://doi.org/10.1101/478255)\]
    \[[doi: 10.1038/s41467-019-12308-0](https://doi.org/10.1038/s41467-019-12308-0)\]
    \[[pdf](https://www.nature.com/articles/s41467-019-12308-0.pdf)\]

12. Bradfield JP<sup>1</sup>, Vogelezang S<sup>1</sup>, …, **Vaudel M**,
    …, Jaddoe VWV<sup>†</sup>, Grant SFA<sup>†,:email:</sup>. *A
    Trans-ancestral Meta-Analysis of Genome-Wide Association Studies
    Reveals Loci Associated with Childhood Obesity*, Human Molecular
    Genetics, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31504550)\]
    \[[doi: 10.1093/hmg/ddz161](https://doi.org/10.1093/hmg/ddz161)\]
    \[[pdf](https://erepo.uef.fi/bitstream/123456789/7860/2/1575979386684725654.pdf)\]

13. Warrington MW<sup>1</sup>, Beaumont R<sup>1</sup>, Horikoshi
    M<sup>1</sup>, Day FR<sup>1</sup>, Helgeland Ø<sup>1</sup>, …,
    **Vaudel M**, …, Njølstad PR, Johansson S, Ong KK<sup>†</sup>,
    McCarthy MI<sup>†</sup>, Perry JRB<sup>†</sup>, Evans
    DM<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>. *Maternal
    and fetal genetic effects on birth weight and their relevance to
    cardio-metabolic risk factors*, Nature Genetics, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31043758)\]
    \[[Preprint](https://doi.org/10.1101/442756)\]
    \[[doi: 10.1038/s41588-019-0403-1](https://doi.org/10.1038/s41588-019-0403-1)\]
    \[[pdf](http://europepmc.org/articles/pmc6522365?pdf=render)\]

14. Zhang-James Y, **Vaudel M**, Mjaavatten O, Berven FS, Haavik J,
    Faraone SV. *Effect of disease-associated SLC9A9 mutations on
    protein-protein interaction networks: implications for molecular
    mechanisms for ADHD and autism*, ADHD Attention Deficit and
    Hyperactivity Disorders, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30927234)\]
    \[[doi: 10.1007/s12402-018-0281-x](https://doi.org/10.1007/s12402-018-0281-x)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

15. Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30251541)\]
    \[[Preprint](https://doi.org/10.1101/333492)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://doi.org/10.1021/acs.jproteome.8b00464)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/22539/4/Burger%2bet%2bal.%2bAnalysing%2bthe%2bstructure%2bof%2bpathways%2band%2bits%2binfluence%2bon%2bthe%2binterpretation%2bof%2bbiomedical%2bdatasets%2b%2528postprint%2529.pdf)\]

16. Barsnes H, **Vaudel M**<sup>:email:</sup>. *SearchGUI: A Highly
    Adaptable Common Interface for Proteomics Search and de Novo
    Engines*, Journal of Proteome Research, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29774740)\]
    \[[doi: 10.1021/acs.jproteome.8b00175](https://doi.org/10.1021/acs.jproteome.8b00175)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/22093/4/Barsnes%2bet%2bal.%2bSearchGUI%2b%2528postprint%2529.pdf)\]

17. Li K, **Vaudel M**, Zhang B, Ren Y, Wen B<sup>:email:</sup>. *PDV:
    an integrative proteomics data viewer*, Bioinformatics, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30169737)\]
    \[[doi: 10.1093/bioinformatics/bty770](https://doi.org/10.1093/bioinformatics/bty770)\]
    \[[pdf](http://europepmc.org/articles/pmc6821182?pdf=render)\]

18. Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>. *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

19. Berland S, Toft Bertelsen TL, Aukrust I, Byška J, **Vaudel M**,
    MacAulay N, Houge G<sup>:email:</sup>. *A de novo Ser111Thr in
    aquaporin-4 in a patient with transient brain edema, transient heart
    failure, intellectual disability and slowly progressive brain
    dysfunction*, Molecular Case Studies, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29437797)\]
    \[[doi: 10.1101/mcs.a002303](https://doi.org/10.1101/mcs.a002303)\]
    \[[pdf](http://molecularcasestudies.cshlp.org/content/4/1/a002303.full.pdf)\]

20. Beaumont RN<sup>1</sup>, Warrington NM<sup>1</sup>, …, **Vaudel M**,
    …, Evans DM<sup>†</sup>, Lawlor DA<sup>†,:email:</sup>, Feenstra
    B<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>. *Genome-wide
    association study of offspring birth weight in 86,577 women
    highlights maternal genetic effects that are independent of fetal
    genetics*, Human Molecular Genetics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29309628)\]
    \[[doi: 10.1101/mcs.a002303](https://doi.org/10.1101/mcs.a002303)\]
    \[[pdf](http://molecularcasestudies.cshlp.org/content/4/1/a002303.full.pdf)\]

21. Chambers MC, Jagtap PD, Johnson JE, McGowan T, Kumar P, Onsongo G,
    Guerrero CR, Barnses H, **Vaudel M**, Martens L, Grüning BA, Cooke
    IR, Heydarian M, Reddy KL, Griffin TJ<sup>:email:</sup>. *An
    accessible proteogenomics informatics resource for cancer
    researchers*, Cancer Research, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29092937)\]
    \[[doi: 10.1158/0008-5472.CAN-17-0331](https://doi.org/10.1158/0008-5472.CAN-17-0331)\]
    \[[pdf](https://cancerres.aacrjournals.org/content/canres/77/21/e43.full.pdf)\]

22. Vizcaíno JA, Meyer G, Perkins SR, Barsnes H, **Vaudel M**,
    Perez-Riverol Y, Terment T, Uszkoreit J, Eineacher M, Fischer L,
    Rappsilber J, Netz E, Walzer M, Kohlbacher O, Leitner A, Chalkley
    RJ, Ghali F, Martínez-Bartolomé S, Deutsch EW, Johnes
    AR<sup>:email:</sup>. *The mzIdentML data standard version 1.2,
    supporting advances in proteome informatics*, Mol Cell Proteomics,
    2017. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28515314)\]
    \[[doi: 10.1074/mcp.M117.068429](https://doi.org/10.1074/mcp.M117.068429)\]
    \[[pdf](https://doi.org/10.1074/mcp.m117.068429)\]

23. Leprevost FdV, Aflitos SA, Grüning BA, Röst HL, Uszkoreit J, Barsnes
    H, **Vaudel M**, Moreno P, Gatto L, Weber J, Bai M, Jimenez RC,
    Sachsenberg T, Pfeufferz J, Alvarez RV, Griss J, Nesvizhskii AI,
    Perez-Riverol Y<sup>:email:</sup>. *BioContainers: An open-source
    and community-driven framework for software standardization*,
    Bioinformatics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28379341)\]
    \[[doi: 10.1093/bioinformatics/btx192](https://doi.org/10.1093/bioinformatics/btx192)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/16/2580/25163480/btx192.pdf)\]

24. Kroksveen AC<sup>:email:</sup>, Guldbrandsen A, **Vaudel M**, Lereim
    RR, Barsnes H, Myr KM, Torkildsen Ø, Berven FS<sup>:email:</sup>.
    *In-Depth Cerebrospinal Fluid Quantitative Proteome and
    Deglycoproteome Analysis: Presenting a Comprehensive Picture of
    Pathways and Processes Affected by Multiple Sclerosis*, Journal of
    Proteome Research, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27728768)\]
    \[[doi: 10.1021/acs.jproteome.6b00659](https://doi.org/10.1021/acs.jproteome.6b00659)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

25. Hernandez-Valladares M<sup>1,:email:</sup>, Aasebø E<sup>1</sup>,
    Mjaavatten O, **Vaudel M**, Bruserud Ø, Berven FS, Selheim F.
    *Reliable FASP-based procedures for optimal quantitative proteomic
    and phosphoproteomic analysis on samples from acute myeloid leukemia
    patients*, Biol Proced Online, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27330413)\]
    \[[doi: 10.1186/s12575-016-0043-0](https://doi.org/10.1186/s12575-016-0043-0)\]
    \[[pdf](https://biologicalproceduresonline.biomedcentral.com/track/pdf/10.1186/s12575-016-0043-0)\]

26. Aasebø E<sup>1</sup>, Mjaavatten O, **Vaudel M**, Farag Y, Selheim
    F, Berven FS, Bruserud Ø, Hernandez-Valladares
    M<sup>1,:email:</sup>. *Freezing effects on the acute myeloid
    leukemia cell proteome and phosphoproteome revealed using optimal
    quantitative workflows*, Journal of Proteomics, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27107777)\]
    \[[doi: 10.1016/j.jprot.2016.03.049](https://doi.org/10.1016/j.jprot.2016.03.049)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

27. Opsahl JA, **Vaudel M**, Guldbrandsen A, Aasebø E, Van Pesch V,
    Franciotta D, Myhr KM, Barsnes H, Berle M, Torkildsen Ø, Kroksveen
    AC, Berven FS<sup>:email:</sup>. *Label free analysis of human
    cerebrospinal fluid addressing various normalization strategies and
    revealing protein groups affected by multiple sclerosis*,
    Proteomics, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26841090)\]
    \[[doi: 10.1002/pmic.201500284](https://doi.org/10.1002/pmic.201500284)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

28. **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H. *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (accepted version)](docs/aam/peptideshaker_submitted.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

29. Verheggen K, Maddelein D, Hulstaert N, Martens L<sup>:email:</sup>,
    Barsnes H, **Vaudel M**. *Pladipus enables universal distributed
    computing in proteomics bioinformatics*, Journal of Proteome
    Research, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26510693)\]
    \[[doi: 10.1021/acs.jproteome.5b00850](https://doi.org/10.1021/acs.jproteome.5b00850)\]
    \[[pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jproteome.5b00850)\]

30. Barsnes H<sup>:email:</sup>, **Vaudel M**, Martens L. *JSparklines:
    Making tabular proteomics data come alive*, Proteomics, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25422159)\]
    \[[doi: 10.1002/pmic.201400356](https://doi.org/10.1002/pmic.201400356)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

31. Bjørlykke Y, Vethe H, **Vaudel M**, Barsnes H, Berven FS, Tjora E,
    Raeder H<sup>:email:</sup>. *Carboxyl-Ester Lipase Maturity-Onset
    Diabetes of the Young Disease Protein Biomarkers in
    Secretin-Stimulated Duodenal Juice*, Journal of Proteome Research,
    2015. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25369532)\]
    \[[doi: 10.1021/pr500750z](https://doi.org/10.1021/pr500750z)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

32. Muth T<sup>1</sup>, Weilnböck L<sup>1</sup>, Rapp E, Huber CG,
    Martens L, **Vaudel M**<sup>:email:</sup>, Barsnes H. *DeNovoGUI: an
    open source graphical user interface for de novo sequencing of
    tandem mass spectra*, Journal of Proteome Research, 2014.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24295440)\]
    \[[doi: 10.1021/pr4008078](https://doi.org/10.1021/pr4008078)\]
    \[[pdf](https://doi.org/10.1021/pr4008078)\]

33. Helgeland E, Breivik LE, **Vaudel M**, Svendsen OS, Garberg H,
    Nordrehaug JE, Berven FS, Jonassen AK<sup>:email:</sup>. *Exploring
    the human plasma proteome for humoral mediators of remote ischemic
    preconditioning - a word of caution*, PLoS One, 2014.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25333471)\]
    \[[doi: 10.1371/journal.pone.0109279](https://doi.org/10.1371/journal.pone.0109279)\]
    \[[pdf](https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0109279&type=printable)\]

34. Vethe H, Finne K, Skogstrand T, **Vaudel M**, Vikse BE, Hultström M,
    Placier S, Scherer A, Tenstad O, Marti HP<sup>:email:</sup>.
    *Distinct protein signature of hypertension-induced damage in the
    renal proteome of the two-kidney, one-clip rat model*, Journal of
    Hypertension, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25304470)\]
    \[[doi: 10.1097/HJH.0000000000000370](https://doi.org/10.1097/HJH.0000000000000370)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

35. Aasebø E, **Vaudel M**, Mjaavatten O, Gaudsdal G, Van der Burgh A,
    Gjertsen BT, Døskeland SO, Brusserud O, Berven FS, Selheim
    F<sup>:email:</sup>. *Performance of super-SILAC based quantitative
    proteomics for comparison of different acute myeloid leukemia (AML)
    cell lines*, Proteomics, 2014.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25044641)\]
    \[[doi: 10.1002/pmic.201300448](https://doi.org/10.1002/pmic.201300448)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

36. **Vaudel M**, Venne S, Berven FS, Zahedi RP, Martens L, Barsnes
    H<sup>:email:</sup>. *Shedding light on black boxes in protein
    identification*, Proteomics, 2014.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24678044)\]
    \[[doi: 10.1002/pmic.201300488](https://doi.org/10.1002/pmic.201300488)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/16924/5/Shedding%20light%20on%20black%20boxes%20in%20protein%20identification%20%28postprint%29.pdf)\]

37. Beck F, Geiger J, Gambaryan S, Veit J, **Vaudel M**, Nollau P,
    Kohlbacher O, Martens L, Walter U, Sickmann A<sup>:email:</sup>,
    Zahedi RP<sup>:email:</sup>. *Time-resolved characterization of
    cAMP/PKA-dependent signaling reveals that platelet inhibition is a
    concerted process involving multiple signaling pathways*, Blood,
    2014. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24324209)\]
    \[[doi: 10.1182/blood-2013-07-512384](https://doi.org/10.1182/blood-2013-07-512384)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

38. **Vaudel M**, Breiter D, Beck F, Rahnenführer J, Martens
    L<sup>:email:</sup>, Zahedi RP. *D-score: a search engine
    independent MD-score*, Proteomics, 2013.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23307401)\]
    \[[doi: 10.1002/pmic.201200408](https://doi.org/10.1002/pmic.201200408)\]
    \[[pdf (accepted
    version)](docs/aam/D-score%20revised%20version.pdf)\] \[[Request a
    reprint](mailto:marc.vaudel@uib.no)\]

39. Burkhart JM<sup>1</sup>, **Vaudel M**<sup>1</sup>, Gambaryan S,
    Radau S, Walter U, Martens L, Geiger J<sup>:email:</sup>, Sickmann
    A, and Zahedi RP<sup>:email:</sup>. *The first comprehensive and
    quantitative analysis of human platelet protein composition allows
    the comparative analysis of structural and functional pathways in
    human platelets*, Blood, 2012.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22869793)\]
    \[[doi: 10.1182/blood-2012-04-416594](https://doi.org/10.1182/blood-2012-04-416594)\]
    \[[pdf](https://ashpublications.org/blood/article-pdf/120/15/e73/1359688/zh804112000e73.pdf)\]

40. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Radau S, Zahedi
    RP, Martens L<sup>:email:</sup>, Sickmann A. *Integral
    Quantification Accuracy estimation for Reporter Ion based
    quantitative proteomics (iQuARI)*, Journal of Proteome Research,
    2012. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22874012)\]
    \[[doi: 10.1021/pr300247u](https://doi.org/10.1021/pr300247u)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

41. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Breiter D, Zahedi
    RP, Sickmann A<sup>:email:</sup>, Martens L. *A complex standard for
    protein identification, designed by evolution*, Journal of Proteome
    Research, 2012.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22489649)\]
    \[[doi: 10.1021/pr300055q](https://doi.org/10.1021/pr300055q)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

42. **Vaudel M**<sup>1</sup>, Barsnes H<sup>1,:email:</sup>, Berven FS,
    Sickmann A, Martens L. *SearchGUI: an open-source graphical user
    interface for simultaneous OMSSA and X\!Tandem searches*,
    Proteomics, 2011.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21337703)\]
    \[[doi: 10.1002/pmic.201000595](https://doi.org/10.1002/pmic.201000595)\]
    \[[pdf (accepted
    version)](docs/aam/Vaudel_et_al_SearchGUI_17092010.pdf)\] \[[Request
    a reprint](mailto:marc.vaudel@uib.no)\]

43. Colaert N, Barsnes H, **Vaudel M**, Helsens K, Timmerman E, Sickmann
    A, Gevaert K, Martens L<sup>:email:</sup>. *Thermo-msf-parser: an
    open source Java library to parse and visualize Thermo Proteome
    Discoverer msf files*, Journal of Proteome Research, 2011.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21714566)\]
    \[[doi: 10.1021/pr2005154](https://doi.org/10.1021/pr2005154)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/16679/3/Colaert%2bet%2bal%2b%2528postprint%2529.pdf)\]

44. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Sickmann A,
    Martens L<sup>:email:</sup>, Zahedi RP. *Peptide identification
    quality control*, Proteomics, 2011.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21500347)\]
    \[[doi: 10.1002/pmic.201000704](https://doi.org/10.1002/pmic.201000704)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

45. Barsnes H, **Vaudel M**, Colaert N, Helsens K, Sickmann A, Berven
    FS, Martens L<sup>:email:</sup>. *compomics-utilities: an
    open-source Java library for computational proteomics*, BMC
    Bioinformatics, 2011.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21385435)\]
    \[[doi: 10.1186/1471-2105-12-70](https://doi.org/10.1186/1471-2105-12-70)\]
    \[[pdf](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-70)\]

46. Burkhart JM<sup>1</sup>, **Vaudel M**<sup>1</sup>, Zahedi RP,
    Martens L, Sickmann A<sup>:email:</sup>. *iTRAQ protein
    quantification: a quality-controlled workflow*, Proteomics, 2011.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21328540)\]
    \[[doi: 10.1002/pmic.201000711](https://doi.org/10.1002/pmic.201000711)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

47. Muth T, **Vaudel M**, Barsnes H, Martens L<sup>:email:</sup>,
    Sickmann A. *XTandem Parser: An open-source library to parse and
    analyse X\!Tandem MS/MS search results*, Proteomics, 2010.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20140905)\]
    \[[doi: 10.1002/pmic.200900759](https://doi.org/10.1002/pmic.200900759)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## Outreach - Proceeding

1.  Burger B<sup>:email:</sup>, **Vaudel M**<sup>:email:</sup>, Barsnes
    H. *Importance of Block Randomization When Designing Proteomics
    Experiments*, Journal of Proteome Research, 2020.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/%2032969222)\]
    \[[doi: 10.1021/acs.jproteome.0c00536](https://doi.org/10.1021/acs.jproteome.0c00536)\]
    \[[pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jproteome.0c00536)\]

2.  Njølstad PR<sup>:email:</sup>, …, **Vaudel M**, …, Stefánsson K.
    *Roadmap for a precision-medicine initiative in the Nordic region*,
    Nature Genetics, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30988515)\]
    \[[doi: 10.1038/s41588-019-0391-1](https://doi.org/10.1038/s41588-019-0391-1)\]
    \[[pdf](https://helda.helsinki.fi/bitstream/10138/325612/1/kaprio_48572_2_merged_1552149424_v2.pdf)\]

3.  Grüning BA, Lampa S, **Vaudel M**, Blankenberg D<sup>:email:</sup>.
    *Software engineering for scientific big data analysis*,
    Gigascience, 2019.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31121028)\]
    \[[doi: 10.1093/gigascience/giz054](https://doi.org/10.1093/gigascience/giz054)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/5/giz054/28698071/giz054.pdf)\]

4.  Willems S, bouyssié D, Deforce D, Dorfer V, Gorshkov V, Kopczynski
    D, Laukens K, Locard-Paulet M, Schwämml V, Uszkoreit J, Valkenborg
    D, **Vaudel M**, Bittremieux W<sup>:email:</sup>. *Proceedings of
    the EuBIC developer’s meeting 2018*, Journal of Proteomics, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29864591)\]
    \[[doi: 10.1016/j.jprot.2018.05.015](https://doi.org/10.1016/j.jprot.2018.05.015)\]
    \[[pdf](https://repository.uantwerpen.be/docman/irua/23a1a2/151764_2019_06_03.pdf)\]

5.  Willems S, Bouyssié D, David M, Locard-Paulet M, Mechtler K,
    Schwämmle V, Uszkoreit J, **Vaudel M**, Dorfer V<sup>:email:</sup>.
    *Proceedings of the EuBIC Winter School 2017*, Journal of
    Proteomics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28385664)\]
    \[[doi: 10.1016/j.jprot.2017.04.001](https://doi.org/10.1016/j.jprot.2017.04.001)\]
    \[[pdf](https://air.unimi.it/bitstream/2434/803373/2/1-s2.0-S2212968519300078-main.pdf)\]

6.  **Vaudel M**, Veit Schwämmle<sup>:email:</sup>, Viktoria Dorfer.
    *The EuPA Bioinformatics Community (EuBIC) initiative*, EuPA Open
    Proteomics, 2016. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/)\]
    \[[doi: 10.1016/j.euprot.2016.03.009](https://doi.org/10.1016/j.euprot.2016.03.009)\]
    \[[pdf](https://doi.org/10.1016/j.euprot.2016.03.009)\]

7.  Guldbrandsen A, **Vaudel M**, Barsnes H<sup>:email:</sup>.
    *Framtiden for biomedisinsk forskning - store endringer og nye
    muligheter*, Best Practice Nordic, 2016.
    [\[Link\]](https://bestprac.no/framtiden-biomedisinsk-forskning-store-endringer-gir-nye-muligheter/)

<br>

## Reviews

1.  Muth T<sup>:email:</sup>, Hartkopf F, **Vaudel
    M**<sup>:email:</sup>, Renard BY. *A Potential Golden Age to
    Come-Current Tools, Recent Use Cases, and Future Avenues for De Novo
    Sequencing in Proteomics*, Proteomics, 2018.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29968278)\]
    \[[doi: 10.1002/pmic.201700150](https://doi.org/10.1002/pmic.201700150)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/20984/3/De%2bnovo%2bsequencing%2breview_revised_final.pdf)\]

2.  Verheggen K, Ræder H, Breven FS, Martens L<sup>:email:</sup>,
    Barsnes H, **Vaudel M**. *Anatomy and evolution of database search
    engines—a central component of mass spectrometry based proteomic
    workflows*, Mass Spectrometry Reviews, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28902424)\]
    \[[doi: 10.1002/mas.21543](https://doi.org/10.1002/mas.21543)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/21005/4/Verheggen%2bet%2bal.%2bProteomic%2bdatabase%2bsearch%2bengines%2b-%2bfinal%2bversion.pdf)\]

3.  Hernandez-Valladares M<sup>:email:</sup>, **Vaudel M**, Selheim F,
    Berven F, Bruserud Ø. *Proteogenomics approaches for studying cancer
    biology and their potential in the identification of Acute Myeloid
    Leukemia biomarkers*, Expert Reviews of Proteomics, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28693350)\]
    \[[doi: 10.1080/14789450.2017.1352474](https://doi.org/10.1080/14789450.2017.1352474)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

4.  **Vaudel M**, Verheggen K, Csordas A, Raeder H, Berven FS, Martens
    L, Vizcaíno JA, Barsnes H<sup>:email:</sup>. *Exploring the
    potential of public proteomics data*, Proteomics, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26449181)\]
    \[[doi: 10.1002/pmic.201500295](https://doi.org/10.1002/pmic.201500295)\]
    \[[pdf](https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/pmic.201500295)\]

5.  **Vaudel M**<sup>:email:</sup>, Barsnes H, Bjerkvig R, Bikfalvi A,
    Selheim F, Berven FS, Daubon T. *Practical considerations for omics
    experiments in biomedical sciences*, Current Pharmaceutical
    Biotechnology, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26278526)\]
    \[[doi: 10.2174/1389201016666150817095348](https://doi.org/10.2174/1389201016666150817095348)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/21252/4/review%2bcpb%2bleukemia%2b07.07.15.pdf)\]

6.  **Vaudel M**<sup>:email:</sup>, Sickmann A, Martens L. *Introduction
    to opportunities and pitfalls in functional mass spectrometry based
    proteomics*, Biochim Biophys Acta, 2014.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23845992)\]
    \[[doi: 10.1016/j.bbapap.2013.06.019](https://doi.org/10.1016/j.bbapap.2013.06.019)\]
    \[[pdf](https://biblio.ugent.be/publication/4259751/file/4286995.pdf)\]

7.  **Vaudel M**, Sickmann A, Martens L<sup>:email:</sup>. *Current
    methods for global proteome identification*, Expert Reviews of
    Proteomics, 2012.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23194269)\]
    \[[doi: 10.1586/epr.12.51](https://doi.org/10.1586/epr.12.51)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

8.  **Vaudel M**<sup>:email:</sup>, Sickmann A, Martens L. *Peptide and
    protein quantification: a map of the minefield*, Proteomics, 2010.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/19953549)\]
    \[[doi: 10.1002/pmic.200900481](https://doi.org/10.1002/pmic.200900481)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## Book Chapters

1.  **Vaudel M**<sup>:email:</sup>. *Proteomics*, Processing
    Metabolomics and Proteomics Data with Open Software: A Practical
    Guide, 2019. \[PubMed\]
    \[[doi: 10.1039/9781788019880-00096](https://doi.org/10.1039/9781788019880-00096)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

2.  Schwämmle V<sup>:email:</sup>, **Vaudel M**. *Computational and
    Statistical Methods for High-Throughput Mass Spectrometry-Based PTM
    Analysis*, Methods in Molecular Biology: Protein Bioinformatics,
    Springer, 2017.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/28150251)\]
    \[[doi: 10.1007/978-1-4939-6783-4\_21](https://doi.org/10.1007/978-1-4939-6783-4_21)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

3.  **Vaudel M**<sup>:email:</sup>. *MS2-Based Quantitation*, Proteome
    Informatics, Royal Society of Chemistry, 2016. \[PubMed\]
    \[[doi: 10.1039/9781782626732-00155](https://doi.org/10.1039/9781782626732-00155)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

4.  Lereim RR, Oveland E, Berven FS, **Vaudel M**<sup>:email:</sup>,
    Barsnes H. *Visualization, Inspection and Interpretation of Shotgun
    Proteomics Identification Results*, Advances in Experimental
    Medicine and Biology: Modern Proteomics – Sample Preparation,
    Analysis and Practical Applications, Springer, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27975220)\]
    \[[doi: 10.1007/978-3-319-41448-5\_11](https://doi.org/10.1007/978-3-319-41448-5_11)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

5.  Muth T, Rapp E, Berven FS, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Tandem Mass Spectrum Sequencing: An
    Alternative to Database Search Engines in Shotgun Proteomics*,
    Advances in Experimental Medicine and Biology: Modern Proteomics –
    Sample Preparation, Analysis and Practical Applications, Springer,
    2016. \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27975219)\]
    \[[doi: 10.1007/978-3-319-41448-5\_10](https://doi.org/10.1007/978-3-319-41448-5_10)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

6.  Verheggen K, Martens L, Berven FS, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Database Search Engines: Paradigms,
    Challenges and Solutions*, Advances in Experimental Medicine and
    Biology: Modern Proteomics – Sample Preparation, Analysis and
    Practical Applications, Springer, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27975215)\]
    \[[doi: 10.1007/978-3-319-41448-5\_6](https://doi.org/10.1007/978-3-319-41448-5_6)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

7.  **Vaudel M**<sup>:email:</sup>, Barsnes H, Ræder H, Berven FS.
    *Using Proteomics Bioinformatics Tools and Resources in
    Proteogenomic Studies*, Advances in Experimental Medicine and
    Biology: Proteogenomics, Springer, 2016.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/27686806)\]
    \[[doi: 10.1007/978-3-319-42316-6\_5](https://doi.org/10.1007/978-3-319-42316-6_5)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

8.  Hernández Sánchez LF, Aasebø E, Selheim F, Berven FS, Ræder H,
    Barsnes H, **Vaudel M**<sup>:email:</sup>. *Systemic Analysis of
    Regulated Functional Networks*, Methods in Molecular Biology:
    Proteomics in Systems Biology, Springer, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26700057)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_21](https://doi.org/10.1007/978-1-4939-3341-9_21)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

9.  Guldbrandsen A, Barsnes H, Kroksveen AC, Berven FS, **Vaudel
    M**<sup>:email:</sup>. *A Simple Workflow for Large Scale Shotgun
    Glycoproteomics*, Methods in Molecular Biology: Proteomics in
    Systems Biology, Springer, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26700056)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_20](https://doi.org/10.1007/978-1-4939-3341-9_20)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

10. Aasebø E, Berven FS, Selheim F, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Interpretation of Quantitative Shotgun
    Proteomic Data*, Methods in Molecular Biology: Proteomics in Systems
    Biology, Springer, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/26700055)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_19](https://doi.org/10.1007/978-1-4939-3341-9_19)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

11. **Vaudel M**<sup>:email:</sup>, Barsnes H, Martens L, Berven FS.
    *Bioinformatics for Proteomics: Opportunities at the interface
    between the scientists, their experiments and the community*,
    Methods in Molecular Biology: Shotgun Proteomics, Springer, 2015.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24791993)\]
    \[[doi: 10.1007/978-1-4939-0685-7\_16](https://doi.org/10.1007/978-1-4939-0685-7_16)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

12. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Zahedi RP,
    Martens L, Sickmann A<sup>:email:</sup>. *iTRAQ data
    interpretation*, Methods in Molecular Biology: Quantitative Methods
    in Proteomics, Springer, 2012.
    \[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22665319)\]
    \[[doi: 10.1007/978-1-61779-885-6\_30](https://doi.org/10.1007/978-1-61779-885-6_30)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## PhD thesis

*Detailed analysis of the blood platelet proteome through the use of
innovative bioinformatics methods for protein identification,
quantification and post-translational modification localization.*
[link](https://hdl.handle.net/1854/LU-3076565) <br>

## About

This document was generated from this [Rmd](README.Rmd). Links to pdfs
are obtained from [Unpaywall](unpaywall.org), queried using the
[roadoi](https://cran.r-project.org/web/packages/roadoi/vignettes/intro.html)
package.

For questions or error report, please do not hesitate to reach out *via*
the [issue tracker](https://github.com/mvaudel/Publications/issues).

For personal use only, no copyright infringement intended. <br>

## R Session Info

``` r
sessionInfo()
```

    ## R version 3.6.1 (2019-07-05)
    ## Platform: x86_64-w64-mingw32/x64 (64-bit)
    ## Running under: Windows 10 x64 (build 17134)
    ## 
    ## Matrix products: default
    ## 
    ## locale:
    ## [1] LC_COLLATE=Norwegian Bokmål_Norway.1252 
    ## [2] LC_CTYPE=Norwegian Bokmål_Norway.1252   
    ## [3] LC_MONETARY=Norwegian Bokmål_Norway.1252
    ## [4] LC_NUMERIC=C                            
    ## [5] LC_TIME=Norwegian Bokmål_Norway.1252    
    ## 
    ## attached base packages:
    ## [1] stats     graphics  grDevices utils     datasets  methods   base     
    ## 
    ## other attached packages:
    ## [1] roadoi_0.6       conflicted_1.0.4
    ## 
    ## loaded via a namespace (and not attached):
    ##  [1] Rcpp_1.0.4.6     pillar_1.4.6     compiler_3.6.1   later_1.1.0.1   
    ##  [5] plyr_1.8.6       tools_3.6.1      digest_0.6.25    jsonlite_1.7.0  
    ##  [9] evaluate_0.14    memoise_1.1.0    lifecycle_0.2.0  tibble_3.0.0    
    ## [13] pkgconfig_2.0.3  rlang_0.4.10     shiny_1.5.0      cli_2.0.2       
    ## [17] curl_4.3         yaml_2.2.1       xfun_0.16        fastmap_1.0.1   
    ## [21] dplyr_1.0.0      stringr_1.4.0    httr_1.4.2       knitr_1.29      
    ## [25] generics_0.0.2   vctrs_0.3.1      tidyselect_1.1.0 glue_1.4.0      
    ## [29] R6_2.4.1         fansi_0.4.1      rmarkdown_2.3    purrr_0.3.4     
    ## [33] tidyr_1.1.0      magrittr_1.5     promises_1.1.1   ellipsis_0.3.0  
    ## [37] htmltools_0.5.0  assertthat_0.2.1 mime_0.9         xtable_1.8-4    
    ## [41] httpuv_1.5.4     stringi_1.4.6    miniUI_0.1.1.1   crayon_1.3.4
