Publications
================
Marc Vaudel
2019-08-01

[![DOI](https://zenodo.org/badge/200110790.svg)](https://zenodo.org/badge/latestdoi/200110790)

This repository lists the scientific publications co-authored by [Marc
Vaudel](https://www.uib.no/en/persons/Marc.Vaudel) as of 26.04.2022.

  - [ORCID: 0000-0003-1179-9578](https://orcid.org/0000-0003-1179-9578)
  - [NIH
    iCite](https://icite.od.nih.gov/analysis?search_id=87lghf7z77q1eloy)
  - [Google
    Scholar](https://scholar.google.com/citations?user=JTK8FTkAAAAJ)

<br>

> <sup>1</sup> shared first, <sup>:email:</sup> correspondence,
> <sup>†</sup> shared last.

<br>

## Five Highlighted Original Articles

1.  Helgeland Ø<sup>1</sup>, **Vaudel M**<sup>1</sup>, Sole-Navais P,
    Flatley C, Juodakis J, Bacelis J, Koløen IL, Peggy Knudsen G,
    Johansson BB, Magnus P, Reichborn Kjennerud T, Juliusson PB,
    Stoltenberg C, Holmen OL, Andreassen OA, Jacobsson B, Njølstad
    PR<sup>:email:</sup>, Johansson S<sup>:email:</sup>.
    *Characterization of the genetic architecture of BMI in infancy and
    early childhood reveals age-specific effects and implicates pathways
    involved in Mendelian obesity*, Nature Metabolism, 2022. \[[PubMed
    id 35315439](https://www.ncbi.nlm.nih.gov/pubmed/35315439)\]
    \[[doi: 10.1038/s42255-022-00549-1](https://doi.org/10.1038/s42255-022-00549-1)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![EarlyGrowth](docs/figures/early_growth.png?raw=true "Early Growth")

</td>

<td>

*<sub>Longitudinal association of BMI with common variation - we
identify different trajectory clusters, time-dependent parent-of-origin
effects, and show how the performance of risk scores evolve during early
growth.</sub>* <br>

</td>

</tr>

</table>

<br>

2.  Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>. *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019. \[[PubMed
    id 31363752](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[Preprint](https://doi.org/10.1101/375097)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

<table>

<tr>

<td width="150">

![PathwayMatcher](docs/figures/PathwayMatcher.png?raw=true
"PathwayMatcher")

</td>

<td>

*<sub>PathwayMatcher can build biological networks of different levels
of granularity using the [Reactome](reactome.org) pathway database, and
map different types of omics data to it. Notably, it can build a network
to the level of
[proteoforms](https://www.nature.com/articles/nmeth.2369), where genes
and proteins are separated according to their post-translational state.
Hence, interactions are more specific, and interactions between
different forms of the same proteins become visible as connections
between different nodes, thus providing a new dimension to study
biological networks.</sub>* <br>

</td>

</tr>

</table>

<br>

3.  Helgeland Ø, **Vaudel M**, Juliusson PB, Lingaas Holmen O, Juodakis
    J, Bacelis J, Jacobsson B, Lindekleiv H, Hveem K, Lie RT, Knudsen
    GP, Stoltenberg C, Magnus P, Sagen JV, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>. *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications, 2019.
    \[[PubMed
    id 31575865](https://www.ncbi.nlm.nih.gov/pubmed/31575865)\]
    \[[Preprint](https://doi.org/10.1101/478255)\]
    \[[doi: 10.1038/s41467-019-12308-0](https://doi.org/10.1038/s41467-019-12308-0)\]
    \[[pdf](https://www.nature.com/articles/s41467-019-12308-0.pdf)\]

<table>

<tr>

<td width="150">

![MH time lapse](docs/figures/mh_time-lapse.gif?raw=true
"MH time lapse")

</td>

<td>

*<sub>The association between genetic variation and phenotypes, *e.g.*
with the Body Mass Index (BMI), is often presented as static, yet it
changes over the course of life. Here we present genome-wide association
results during the first years of life, from birth to age 8, making
possibly one of the first genetic time lapse. It represents 12 GWASs on
the [MoBa
cohort](https://www.fhi.no/studier/moba/forskere/sporreskjemaer---mor-og-barn-unders/),
for which we built a complete bioinformatic environment from
scratch.</sub>* <br>

</td>

</tr>

</table>

<br>

4.  Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>. *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017. \[[PubMed
    id 28334306](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

<table>

<tr>

<td width="150">

![PeptideMapper](docs/figures/PeptideMapper.png?raw=true
"PeptideMapper")

</td>

<td>

*<sub>PeptideMapper is an efficient sequence alignment tool for
proteomics. Sequences can be partial with missing sections represented
as mass gaps or wild cards. It accounts for post-translational
modifications and sequence variants during mapping.</sub>* <br>

</td>

</tr>

</table>

<br>

5.  **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H. *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015. \[[PubMed
    id 25574629](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (accepted version)](docs/aam/peptideshaker_submitted.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

<table>

<tr>

<td width="150">

![PeptideShaker](docs/figures/SearchGUI_and_PS-overview-figure.png?raw=true
"PeptideShaker")

</td>

<td>

*<sub>PeptideShaker is a bioinformatic tools that processes proteomics
identification results. It has both rich graphical and command line
interfaces and can operate on in-house data just like on public
proteomics data from the [EMBL-EBI
PRIDE](https://www.ebi.ac.uk/pride/archive/) repository. It can handle
results from database searches as well as *de novo* sequencing, with
support for sequence variants.</sub>* <br>

</td>

</tr>

</table>

<br>

## Original Articles

1.  Burger B<sup>1</sup>, **Vaudel M**, Barsnes H. *Automated splitting
    into batches for observational biomedical studies with sequential
    processing*, Biostatistics, 2022. (*in press*)

2.  Helgeland Ø<sup>1</sup>, **Vaudel M**<sup>1</sup>, Sole-Navais P,
    Flatley C, Juodakis J, Bacelis J, Koløen IL, Knudsen GP, Johansson
    BB, Magnus P, Kjennerud TR, Juliusson PB, Stoltenberg C, Holmen OL,
    Andreassen OA, Jacobsson B, Njølstad PR<sup>:email:</sup>, Johansson
    S<sup>:email:</sup>. *Characterization of the genetic architecture
    of infant and early childhood body mass index*, Nature Metabolism,
    2022. \[[PubMed
    id 35315439](https://www.ncbi.nlm.nih.gov/pubmed/35315439)\]
    \[[doi: 10.1038/s42255-022-00549-1](https://doi.org/10.1038/s42255-022-00549-1)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

3.  Carrasco M, Wang C, Søviknes AM, Bjørlykke Y, Abadpour S, Paulo JA,
    Tjora E, Njølstad P, Ghabayen J, Nermoen I, Lyssenko V, Chera S,
    Ghila LM, **Vaudel M**, Scholz H, Ræder H<sup>:email:</sup>.
    *Spatial Environment Affects HNF4A Mutation-Specific Proteome
    Signatures and Cellular Morphology in hiPSC-Derived β-Like Cells*,
    Diabetes, 2022. \[[PubMed
    id 35043148](https://www.ncbi.nlm.nih.gov/pubmed/35043148)\]
    \[[doi: 10.2337/db20-1279](https://doi.org/10.2337/db20-1279)\]
    \[[pdf](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8965667)\]

4.  Flatley C<sup>:email:</sup>, Sole-Navais P, **Vaudel M**, Helgeland
    Ø, Modzelewska D, Johansson S, Jacobsson B<sup>†</sup>, Njølstad
    P<sup>†</sup>. *Placental weight centiles adjusted for age, parity
    and fetal sex*, Placenta, 2022. \[[PubMed
    id 34773745](https://www.ncbi.nlm.nih.gov/pubmed/34773745)\]
    \[[doi: 10.1016/j.placenta.2021.10.011](https://doi.org/10.1016/j.placenta.2021.10.011)\]
    \[[pdf](https://doi.org/10.1016/j.placenta.2021.10.011)\]

5.  Farag YM, Horro C, **Vaudel M**, Barsnes H<sup>:email:</sup>.
    *PeptideShaker Online: A User-Friendly Web-Based Framework for the
    Identification of Mass Spectrometry-Based Proteomics Data*, Journal
    of Proteome Research, 2021. \[[PubMed
    id 34709836](https://www.ncbi.nlm.nih.gov/pubmed/34709836)\]
    \[[doi: 10.1021/acs.jproteome.1c00678](https://doi.org/10.1021/acs.jproteome.1c00678)\]
    \[[pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jproteome.1c00678)\]

6.  Dai C, Füllgrabe A, Pfeuffer J, Solovyeva EM, Deng J, Moreno P,
    Kamatchinathan S, Kundu DJ, George N, Fexova S, Grüning B, Föll MC,
    Griss J, **Vaudel M**, Audain E, Locard-Paulet M, Turewicz M,
    Eisenacher M, Uszkoreit J, Van Den Bossche T, Schwämmle V, Webel H,
    Schulze S, Bouyssié D, Jayaram S, Duggineni VK, Samaras P, Wilhelm
    M, Choi M, Wang M, Kohlbacher O, Brazma A, Papatheodorou I, Bandeira
    N, Deutsch EW, Vizcaíno JA, Bai M<sup>:email:</sup>, Sachsenberg
    T<sup>:email:</sup>, Levitsky LI<sup>:email:</sup>, Perez-Riverol
    Y<sup>:email:</sup>. *A proteomics sample metadata representation
    for multiomics integration and big data analysis*, Nature
    Communications, 2021. \[[PubMed
    id 34615866](https://www.ncbi.nlm.nih.gov/pubmed/34615866)\]
    \[[doi: 10.1038/s41467-021-26111-3](https://doi.org/10.1038/s41467-021-26111-3)\]
    \[[pdf](https://www.nature.com/articles/s41467-021-26111-3.pdf)\]

7.  Eriksson D<sup>1</sup>, Røyrvik EC<sup>1</sup>, Aranda-Guillén
    M<sup>1</sup>, Berger AH, Landegren N, Artaza H, Hallgren Å, Grytaas
    MA, Ström S, Bratland E, Botusan IR, Oftedal BE, Breivik L, **Vaudel
    M**, Helgeland Ø, Falorni A, Jørgensen AP, Hulting A-L, Svartberg J,
    Ekwall O, Fougner KJ, Wahlberg J, Nedrebø BG, Dahlqvist P, The
    Norwegian Addison Registry Study Group, The Swedish Addison Registry
    Study Group, Knappskog PM, Bøe AS, Bensing S, Johansson
    S<sup>†</sup>, Kämpe O<sup>†,:email:</sup>, Husebye
    ES<sup>†,:email:</sup>. *GWAS for autoimmune Addison’s disease
    identifies multiple risk loci and highlights AIRE in disease
    susceptibility*, Nature Communications, 2021. \[[PubMed
    id 33574239](https://www.ncbi.nlm.nih.gov/pubmed/33574239)\]
    \[[doi: 10.1038/s41467-021-21015-8](https://doi.org/10.1038/s41467-021-21015-8)\]
    \[[pdf](https://www.nature.com/articles/s41467-021-21015-8.pdf)\]

8.  Sole-Navais P<sup>:email:</sup>, Bacelis J, Helgeland Ø, Modzelewska
    D, **Vaudel M**, Flatley C, Andreassen O, Njølstad PR, Muglia LJ,
    Johansson S, Zhang G, Jacobsson B. *Autozygosity mapping and
    time-to-spontaneous delivery in Norwegian parent-offspring trios*,
    Human Molecular Genetics, 2020. \[[PubMed
    id 33291140](https://www.ncbi.nlm.nih.gov/pubmed/33291140)\]
    \[[doi: 10.1093/hmg/ddaa255](https://doi.org/10.1093/hmg/ddaa255)\]
    \[[pdf](https://academic.oup.com/hmg/article-pdf/29/23/3845/36194738/ddaa255.pdf)\]

9.  Gupta MK, Vethe H, Softic S, Rao TN, Wagh V, Shirakawa J, Barsnes H,
    **Vaudel M**, Takatani T, Kahraman S, Sakaguchi M, Martinez R, Hu J,
    Bjørlykke Y, Raeder H, Kulkarni RN<sup>:email:</sup>. *Leptin
    Receptor Signaling Regulates Protein Synthesis Pathways and Neuronal
    Differentiation in Pluripotent Stem Cells*, Stem Cell Reports, 2020.
    \[[PubMed
    id 33125875](https://www.ncbi.nlm.nih.gov/pubmed/33125875)\]
    \[[doi: 10.1016/j.stemcr.2020.10.001](https://doi.org/10.1016/j.stemcr.2020.10.001)\]
    \[[pdf](http://www.cell.com/article/S2213671120303866/pdf)\]

10. Vogelezang S<sup>1</sup>, Bradfield JP<sup>1</sup>, Ahluwalia TS,
    Curtin JA, Lakka TA, Grarup N, Scholz M, van der Most PJ, Monnereau
    C, Stergiakouli E, Heiskala A, Horikoshi M, Fedko IO, Vilor-Tejedor
    N, Cousminer DL, Standl M, Wang CA, Viikari J, Geller F, Íñiguez C,
    Pitkänen N, Chesi A, Bacelis J, Yengo L, Torrent M, Ntalla I,
    Helgeland Ø, Selzam S, Vonk JM, Zafarmand MH, Heude B, Farooqi IS,
    Alyass A, Beaumont RN, Have CT, Rzehak P, Bilbao JR, Schnurr TM,
    Barroso I, Bønnelykke K, Beilin LJ, Carstensen L, Charles MA, Chawes
    B, Clément K, Closa-Monasterolo R, Custovic A, Eriksson JG,
    Escribano J, Groen-Blokhuis M, Grote V, Gruszfeld D, Hakonarson H,
    Hansen T, Hattersley AT, Hollensted M, Hottenga JJ, Hyppönen E,
    Johansson S, Joro R, Kähönen M, Karhunen V, Kiess W, Knight BA,
    Koletzko B, Kühnapfel A, Landgraf K, Langhendries JP, Lehtimäki T,
    Leinonen JT, Li A, Lindi V, Lowry E, Bustamante M, Medina-Gomez C,
    Melbye M, Michaelsen KF, Morgen CS, Mori TA, Nielsen TRH, Niinikoski
    H, Oldehinkel AJ, Pahkala K, Panoutsopoulou K, Pedersen O, Pennell
    CE, Power C, Reijneveld SA, Rivadeneira F, Simpson A, Sly PD,
    Stokholm J, Teo KK, Thiering E, Timpson NJ, Uitterlinden AG, van
    Beijsterveldt CEM, van Schaik BDC, **Vaudel M**, Verduci E, Vinding
    RK, Vogel M, Zeggini E, Sebert S, Lind MV, Brown CD, Santa-Marina L,
    Reischl E, Frithioff-Bøjsøe C, Meyre D, Wheeler E, Ong K, Nohr EA,
    Vrijkotte TGM, Koppelman GH, Plomin R, Njølstad PR, Dedoussis GD,
    Froguel P, Sørensen TIA, Jacobsson B, Freathy RM, Zemel BS,
    Raitakari O, Vrijheid M, Feenstra B, Lyytikäinen LP, Snieder H,
    Kirsten H, Holt PG, Heinrich J, Widén E, Sunyer J, Boomsma DI,
    Järvelin MR, Körner A, Davey Smith G, Holm JC, Atalay M, Murray C,
    Bisgaard H, McCarthy MI; Early Growth Genetics Consortium, Jaddoe
    VWV<sup>†</sup>, Grant SFA<sup>†</sup>, Felix
    JF<sup>†,:email:</sup>. *Novel loci for childhood body mass index
    and shared heritability with adult cardiometabolic traits*, PLoS
    Genetics, 2020. \[[PubMed
    id 33045005](https://www.ncbi.nlm.nih.gov/pubmed/33045005)\]
    \[[doi: 10.1371/journal.pgen.1008718](https://doi.org/10.1371/journal.pgen.1008718)\]
    \[[pdf](https://journals.plos.org/plosgenetics/article/file?id=10.1371/journal.pgen.1008718&type=printable)\]

11. Smajlagić D, Lavrichenko K, Berland S, Helgeland Ø, Knudsen GP,
    **Vaudel M**, Haavik J, Knappskog PM, Njølstad PR, Houge G,
    Johansson S<sup>:email:</sup>. *Population prevalence and
    inheritance pattern of recurrent CNVs associated with
    neurodevelopmental disorders in 12,252 newborns and their parents*,
    European Journal of Human Genetics, 2020. \[[PubMed
    id 32778765](https://www.ncbi.nlm.nih.gov/pubmed/32778765)\]
    \[[doi: 10.1038/s41431-020-00707-7](https://doi.org/10.1038/s41431-020-00707-7)\]
    \[[pdf](https://www.nature.com/articles/s41431-020-00707-7.pdf)\]

12. Aasebø E, Berven FS, Bartaula-Brevik S, Stokowy T, Hovland R,
    **Vaudel M**, Døskeland SO, McCormack E, Batth TS, Olsen JV,
    Bruserud Ø, Selheim F<sup>†</sup>, Hernandez-Valladares
    M<sup>†,:email:</sup>. *Proteome and Phosphoproteome Changes
    Associated with Prognosis in Acute Myeloid Leukemia*, Cancers, 2020.
    \[[PubMed
    id 32192169](https://www.ncbi.nlm.nih.gov/pubmed/32192169)\]
    \[[doi: 10.3390/cancers12030709](https://doi.org/10.3390/cancers12030709)\]
    \[[pdf](https://www.mdpi.com/2072-6694/12/3/709/pdf)\]

13. Loo LSW, Vethe H, Soetedjo AAP, Paulo JA, Jasmen J, Jackson N,
    Bjørlykke Y, Valdez IA, **Vaudel M**, Barsnes H, Gygi SP, Ræder H,
    Teo AKK<sup>:email:</sup>, Kulkarni RN<sup>:email:</sup>. *Dynamic
    proteome profiling of human pluripotent stem cell-derived pancreatic
    progenitors*, Stem Cells, 2019. \[[PubMed
    id 31828876](https://www.ncbi.nlm.nih.gov/pubmed/31828876)\]
    \[[doi: 10.1002/stem.3135](https://doi.org/10.1002/stem.3135)\]
    \[[pdf](https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/stem.3135)\]

14. Bjørlykke Y, Søviknes AM, Hoareau L, Vethe H, Mathisen AF, Chera S,
    **Vaudel M**, Ghila L, Ræder H<sup>:email:</sup>. *Reprogrammed
    cells display distinct proteomic signatures associated with colony
    morphology variability*, Stem Cells International, 2019. \[[PubMed
    id 31827534](https://www.ncbi.nlm.nih.gov/pubmed/31827534)\]
    \[[doi: 10.1155/2019/8036035](https://doi.org/10.1155/2019/8036035)\]
    \[[pdf](https://doi.org/10.1155/2019/8036035)\]

15. Qiao Z, Zheng J, Helgeland Ø, **Vaudel M**, Johansson S, Njølstad
    PR, Davey Smith G, Warrington NM, Evans D<sup>:email:</sup>.
    *Introducing M-GCTA a software package to estimate maternal (or
    paternal) genetic effects on offspring phenotypes*, Behavior
    Genetics, 2019. \[[PubMed
    id 31493278](https://www.ncbi.nlm.nih.gov/pubmed/31493278)\]
    \[[doi: 10.1007/s10519-019-09969-4](https://doi.org/10.1007/s10519-019-09969-4)\]
    \[[pdf](https://espace.library.uq.edu.au/view/UQ:2ce0c5e/M-GCTA_R1_Accepted.pdf)\]

16. Hernández Sánchez LF, Burger B, Horro C, Fabregat A, Johansson S,
    Njølstad PR, Barsnes H, Hermjakob H, **Vaudel
    M**<sup>:email:</sup>. *PathwayMatcher: proteoform-centric network
    construction enables fine-granularity multi-omics pathway mapping*,
    GigaScience, 2019. \[[PubMed
    id 31363752](https://www.ncbi.nlm.nih.gov/pubmed/31363752)\]
    \[[Preprint](https://doi.org/10.1101/375097)\]
    \[[doi: 10.1093/gigascience/giz088](https://doi.org/10.1093/gigascience/giz088)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/8/giz088/29020604/giz088.pdf)\]

17. Helgeland Ø, **Vaudel M**, Juliusson PB, Lingaas Holmen O, Juodakis
    J, Bacelis J, Jacobsson B, Lindekleiv H, Hveem K, Lie RT, Knudsen
    GP, Stoltenberg C, Magnus P, Sagen JV, Molven A, Johansson
    S<sup>†:email:</sup>, Njølstad PR<sup>†:email:</sup>. *Genome-wide
    association study reveals a dynamic role of common genetic variation
    in infant and early childhood growth*, Nature Communications, 2019.
    \[[PubMed
    id 31575865](https://www.ncbi.nlm.nih.gov/pubmed/31575865)\]
    \[[Preprint](https://doi.org/10.1101/478255)\]
    \[[doi: 10.1038/s41467-019-12308-0](https://doi.org/10.1038/s41467-019-12308-0)\]
    \[[pdf](https://www.nature.com/articles/s41467-019-12308-0.pdf)\]

18. Bradfield JP<sup>1</sup>, Vogelezang S<sup>1</sup>, Felix JF, Chesi
    A, Helgeland Ø, Horikoshi M, Karhunen V, Lowry E, Cousminer DL,
    Ahluwalia TS, Thiering E, Boh ET, Zafarmand MH, Vilor-Tejedor N,
    Wang CA, Joro R, Chen Z, Gauderman WJ, Pitkänen N, Parra EJ,
    Fernandez-Rhodes L, Alyass A, Monnereau C, Curtin JA, Have CT,
    McCormack SE, Hollensted M, Frithioff-Bøjsøe C, Valladares-Salgado
    A, Peralta-Romero J, Teo YY, Standl M, Leinonen JT, Holm JC, Peters
    T, Vioque J, Vrijheid M, Simpson A, Custovic A, **Vaudel M**,
    Canouil M, Lindi V, Atalay M, Kähönen M, Raitakari OT, van Schaik
    BDC, Berkowitz RI, Cole SA, Voruganti VS, Wang Y, Highland HM,
    Comuzzie AG, Butte NF, Justice AE, Gahagan S, Blanco E, Lehtimäki T,
    Lakka TA, Hebebrand J, Bonnefond A, Grarup N, Froguel P, Lyytikäinen
    LP, Cruz M, Kobes S, Hanson RL, Zemel BS, Hinney A, Teo KK, Meyre D,
    North KE, Gilliland FD, Bisgaard H, Bustamante M, Bonnelykke K,
    Pennell CE, Rivadeneira F, Uitterlinden AG, Baier LJ, Vrijkotte TGM,
    Heinrich J, Sørensen TIA, Saw SM, Pedersen O, Hansen T, Eriksson J,
    Widén E, McCarthy MI, Njølstad PR, Power C, Hyppönen E, Sebert S,
    Brown CD, Järvelin MR, Timpson NJ, Johansson S, Hakonarson H, Jaddoe
    VWV<sup>†</sup>; Early Growth Genetics Consortium, Grant
    SFA<sup>†,:email:</sup>. *A Trans-ancestral Meta-Analysis of
    Genome-Wide Association Studies Reveals Loci Associated with
    Childhood Obesity*, Human Molecular Genetics, 2019. \[[PubMed
    id 31504550](https://www.ncbi.nlm.nih.gov/pubmed/31504550)\]
    \[[doi: 10.1093/hmg/ddz161](https://doi.org/10.1093/hmg/ddz161)\]
    \[[pdf](https://pureadmin.qub.ac.uk/ws/files/176973914/ddz161.pdf)\]

19. Warrington NM<sup>1</sup>, Beaumont RN<sup>1</sup>, Horikoshi
    M<sup>1</sup>, Day FR<sup>1</sup>, Helgeland Ø<sup>1</sup>, Laurin
    C, Bacelis J, Peng S, Hao K, Feenstra B, Wood AR, Mahajan A, Tyrrell
    J, Robertson NR, Rayner NW, Qiao Z, Moen GH, **Vaudel M**, Marsit
    CJ, Chen J, Nodzenski M, Schnurr TM, Zafarmand MH, Bradfield JP,
    Grarup N, Kooijman MN, Li-Gao R, Geller F, Ahluwalia TS, Paternoster
    L, Rueedi R, Huikari V, Hottenga JJ, Lyytikäinen LP, Cavadino A,
    Metrustry S, Cousminer DL, Wu Y, Thiering E, Wang CA, Have CT,
    Vilor-Tejedor N, Joshi PK, Painter JN, Ntalla I, Myhre R, Pitkänen
    N, van Leeuwen EM, Joro R, Lagou V, Richmond RC, Espinosa A, Barton
    SJ, Inskip HM, Holloway JW, Santa-Marina L, Estivill X, Ang W, Marsh
    JA, Reichetzeder C, Marullo L, Hocher B, Lunetta KL, Murabito JM,
    Relton CL, Kogevinas M, Chatzi L, Allard C, Bouchard L, Hivert MF,
    Zhang G, Muglia LJ, Heikkinen J; EGG Consortium, Morgen CS, van
    Kampen AHC, van Schaik BDC, Mentch FD, Langenberg C, Luan J, Scott
    RA, Zhao JH, Hemani G, Ring SM, Bennett AJ, Gaulton KJ,
    Fernandez-Tajes J, van Zuydam NR, Medina-Gomez C, de Haan HG,
    Rosendaal FR, Kutalik Z, Marques-Vidal P, Das S, Willemsen G, Mbarek
    H, Müller-Nurasyid M, Standl M, Appel EVR, Fonvig CE, Trier C, van
    Beijsterveldt CEM, Murcia M, Bustamante M, Bonas-Guarch S, Hougaard
    DM, Mercader JM, Linneberg A, Schraut KE, Lind PA, Medland SE,
    Shields BM, Knight BA, Chai JF, Panoutsopoulou K, Bartels M, Sánchez
    F, Stokholm J, Torrents D, Vinding RK, Willems SM, Atalay M, Chawes
    BL, Kovacs P, Prokopenko I, Tuke MA, Yaghootkar H, Ruth KS, Jones
    SE, Loh PR, Murray A, Weedon MN, Tönjes A, Stumvoll M, Michaelsen
    KF, Eloranta AM, Lakka TA, van Duijn CM, Kiess W, Körner A,
    Niinikoski H, Pahkala K, Raitakari OT, Jacobsson B, Zeggini E,
    Dedoussis GV, Teo YY, Saw SM, Montgomery GW, Campbell H, Wilson JF,
    Vrijkotte TGM, Vrijheid M, de Geus EJCN, Hayes MG, Kadarmideen HN,
    Holm JC, Beilin LJ, Pennell CE, Heinrich J, Adair LS, Borja JB,
    Mohlke KL, Eriksson JG, Widén EE, Hattersley AT, Spector TD, Kähönen
    M, Viikari JS, Lehtimäki T, Boomsma DI, Sebert S, Vollenweider P,
    Sørensen TIA, Bisgaard H, Bønnelykke K, Murray JC, Melbye M, Nohr
    EA, Mook-Kanamori DO, Rivadeneira F, Hofman A, Felix JF, Jaddoe VWV,
    Hansen T, Pisinger C, Vaag AA, Pedersen O, Uitterlinden AG, Järvelin
    MR, Power C, Hyppönen E, Scholtens DM, Lowe WL Jr, Davey Smith G,
    Timpson NJ, Morris AP, Wareham NJ, Hakonarson H, Grant SFA, Frayling
    TM, Lawlor DA, Njølstad PR, Johansson S, Ong KK<sup>†</sup>,
    McCarthy MI<sup>†</sup>, Perry JRB<sup>†</sup>, Evans
    DM<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>. *Maternal
    and fetal genetic effects on birth weight and their relevance to
    cardio-metabolic risk factors*, Nature Genetics, 2019. \[[PubMed
    id 31043758](https://www.ncbi.nlm.nih.gov/pubmed/31043758)\]
    \[[Preprint](https://doi.org/10.1101/442756)\]
    \[[doi: 10.1038/s41588-019-0403-1](https://doi.org/10.1038/s41588-019-0403-1)\]
    \[[pdf](https://europepmc.org/articles/pmc6522365?pdf=render)\]

20. Zhang-James Y, **Vaudel M**, Mjaavatten O, Berven FS, Haavik J,
    Faraone SV. *Effect of disease-associated SLC9A9 mutations on
    protein-protein interaction networks: implications for molecular
    mechanisms for ADHD and autism*, ADHD Attention Deficit and
    Hyperactivity Disorders, 2019. \[[PubMed
    id 30927234](https://www.ncbi.nlm.nih.gov/pubmed/30927234)\]
    \[[doi: 10.1007/s12402-018-0281-x](https://doi.org/10.1007/s12402-018-0281-x)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

21. Burger B, Hernández Sánchez LF, Lereim RR, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Analyzing the Structure of Pathways and Its
    Influence on the Interpretation of Biomedical Proteomics Data Sets*,
    Journal of Proteome Research, 2018. \[[PubMed
    id 30251541](https://www.ncbi.nlm.nih.gov/pubmed/30251541)\]
    \[[Preprint](https://doi.org/10.1101/333492)\]
    \[[doi: 10.1021/acs.jproteome.8b00464](https://doi.org/10.1021/acs.jproteome.8b00464)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/22539/4/Burger%2bet%2bal.%2bAnalysing%2bthe%2bstructure%2bof%2bpathways%2band%2bits%2binfluence%2bon%2bthe%2binterpretation%2bof%2bbiomedical%2bdatasets%2b%2528postprint%2529.pdf)\]

22. Barsnes H, **Vaudel M**<sup>:email:</sup>. *SearchGUI: A Highly
    Adaptable Common Interface for Proteomics Search and de Novo
    Engines*, Journal of Proteome Research, 2018. \[[PubMed
    id 29774740](https://www.ncbi.nlm.nih.gov/pubmed/29774740)\]
    \[[doi: 10.1021/acs.jproteome.8b00175](https://doi.org/10.1021/acs.jproteome.8b00175)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/22093/4/Barsnes%2bet%2bal.%2bSearchGUI%2b%2528postprint%2529.pdf)\]

23. Li K, **Vaudel M**, Zhang B, Ren Y, Wen B<sup>:email:</sup>. *PDV:
    an integrative proteomics data viewer*, Bioinformatics, 2018.
    \[[PubMed
    id 30169737](https://www.ncbi.nlm.nih.gov/pubmed/30169737)\]
    \[[doi: 10.1093/bioinformatics/bty770](https://doi.org/10.1093/bioinformatics/bty770)\]
    \[[pdf](https://europepmc.org/articles/pmc6821182?pdf=render)\]

24. Kopczynski D, Barsnes H, Njølstad PR, Sickmann A, **Vaudel
    M**<sup>:email:</sup>, Ahrends R<sup>:email:</sup>. *PeptideMapper:
    Efficient and Versatile Amino Acid Sequence and Tag Mapping*,
    Bioinformatics, 2017. \[[PubMed
    id 28334306](https://www.ncbi.nlm.nih.gov/pubmed/28334306)\]
    \[[doi: 10.1093/bioinformatics/btx122](https://doi.org/10.1093/bioinformatics/btx122)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/13/2042/25155905/btx122.pdf)\]

25. Berland S, Toft Bertelsen TL, Aukrust I, Byška J, **Vaudel M**,
    MacAulay N, Houge G<sup>:email:</sup>. *A de novo Ser111Thr in
    aquaporin-4 in a patient with transient brain edema, transient heart
    failure, intellectual disability and slowly progressive brain
    dysfunction*, Molecular Case Studies, 2017. \[[PubMed
    id 29437797](https://www.ncbi.nlm.nih.gov/pubmed/29437797)\]
    \[[doi: 10.1101/mcs.a002303](https://doi.org/10.1101/mcs.a002303)\]
    \[[pdf](http://molecularcasestudies.cshlp.org/content/4/1/a002303.full.pdf)\]

26. Beaumont RN<sup>1</sup>, Warrington NM<sup>1</sup>, Cavadino A,
    Tyrrell J, Nodzenski M, Horikoshi M, Geller F, Myhre R, Richmond RC,
    Paternoster L, Bradfield JP, Kreiner-Møller E, Huikari V, Metrustry
    S, Lunetta KL, Painter JN, Hottenga JJ, Allard C, Barton SJ,
    Espinosa A, Marsh JA, Potter C, Zhang G, Ang W, Berry DJ, Bouchard
    L, Das S; Early Growth Genetics (EGG) Consortium, Hakonarson H,
    Heikkinen J, Helgeland Ø, Hocher B, Hofman A, Inskip HM, Jones SE,
    Kogevinas M, Lind PA, Marullo L, Medland SE, Murray A, Murray JC,
    Njølstad PR, Nohr EA, Reichetzeder C, Ring SM, Ruth KS,
    Santa-Marina L, Scholtens DM, Sebert S, Sengpiel V, Tuke MA,
    **Vaudel M**, Weedon MN, Willemsen G, Wood AR, Yaghootkar H, Muglia
    LJ, Bartels M, Relton CL, Pennell CE, Chatzi L, Estivill X, Holloway
    JW, Boomsma DI, Montgomery GW, Murabito JM, Spector TD, Power C,
    Järvelin MR, Bisgaard H, Grant SFA, Sørensen TIA, Jaddoe VW,
    Jacobsson B, Melbye M, McCarthy MI, Hattersley AT, Hayes MG,
    Frayling TM, Hivert MF, Felix JF, Hyppönen E, Lowe WL Jr, Evans
    DM<sup>†</sup>, Lawlor DA<sup>†,:email:</sup>, Feenstra
    B<sup>†,:email:</sup>, Freathy RM<sup>†,:email:</sup>..
    *Genome-wide association study of offspring birth weight in 86,577
    women highlights maternal genetic effects that are independent of
    fetal genetics*, Human Molecular Genetics, 2017. \[[PubMed
    id 29309628](https://www.ncbi.nlm.nih.gov/pubmed/29309628)\]
    \[[doi: 10.1101/mcs.a002303](https://doi.org/10.1101/mcs.a002303)\]
    \[[pdf](http://molecularcasestudies.cshlp.org/content/4/1/a002303.full.pdf)\]

27. Chambers MC, Jagtap PD, Johnson JE, McGowan T, Kumar P, Onsongo G,
    Guerrero CR, Barnses H, **Vaudel M**, Martens L, Grüning BA, Cooke
    IR, Heydarian M, Reddy KL, Griffin TJ<sup>:email:</sup>. *An
    accessible proteogenomics informatics resource for cancer
    researchers*, Cancer Research, 2017. \[[PubMed
    id 29092937](https://www.ncbi.nlm.nih.gov/pubmed/29092937)\]
    \[[doi: 10.1158/0008-5472.CAN-17-0331](https://doi.org/10.1158/0008-5472.CAN-17-0331)\]
    \[[pdf](https://cancerres.aacrjournals.org/content/canres/77/21/e43.full.pdf)\]

28. Vizcaíno JA, Meyer G, Perkins SR, Barsnes H, **Vaudel M**,
    Perez-Riverol Y, Terment T, Uszkoreit J, Eineacher M, Fischer L,
    Rappsilber J, Netz E, Walzer M, Kohlbacher O, Leitner A, Chalkley
    RJ, Ghali F, Martínez-Bartolomé S, Deutsch EW, Johnes
    AR<sup>:email:</sup>. *The mzIdentML data standard version 1.2,
    supporting advances in proteome informatics*, Mol Cell Proteomics,
    2017. \[[PubMed
    id 28515314](https://www.ncbi.nlm.nih.gov/pubmed/28515314)\]
    \[[doi: 10.1074/mcp.M117.068429](https://doi.org/10.1074/mcp.M117.068429)\]
    \[[pdf](https://doi.org/10.1074/mcp.m117.068429)\]

29. Leprevost FdV, Aflitos SA, Grüning BA, Röst HL, Uszkoreit J, Barsnes
    H, **Vaudel M**, Moreno P, Gatto L, Weber J, Bai M, Jimenez RC,
    Sachsenberg T, Pfeufferz J, Alvarez RV, Griss J, Nesvizhskii AI,
    Perez-Riverol Y<sup>:email:</sup>. *BioContainers: An open-source
    and community-driven framework for software standardization*,
    Bioinformatics, 2017. \[[PubMed
    id 28379341](https://www.ncbi.nlm.nih.gov/pubmed/28379341)\]
    \[[doi: 10.1093/bioinformatics/btx192](https://doi.org/10.1093/bioinformatics/btx192)\]
    \[[pdf](https://academic.oup.com/bioinformatics/article-pdf/33/16/2580/25163480/btx192.pdf)\]

30. Kroksveen AC<sup>:email:</sup>, Guldbrandsen A, **Vaudel M**, Lereim
    RR, Barsnes H, Myr KM, Torkildsen Ø, Berven FS<sup>:email:</sup>.
    *In-Depth Cerebrospinal Fluid Quantitative Proteome and
    Deglycoproteome Analysis: Presenting a Comprehensive Picture of
    Pathways and Processes Affected by Multiple Sclerosis*, Journal of
    Proteome Research, 2017. \[[PubMed
    id 27728768](https://www.ncbi.nlm.nih.gov/pubmed/27728768)\]
    \[[doi: 10.1021/acs.jproteome.6b00659](https://doi.org/10.1021/acs.jproteome.6b00659)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/23324/4/Kroksveen%2bet%2bal.%2bIn-depth%2bcerebrospinal%2bfluid%2bquantitative%2bproteome%2band%2bdeglycoproteome%2banalysis%2b%2528postprint%2529.pdf)\]

31. Hernandez-Valladares M<sup>1,:email:</sup>, Aasebø E<sup>1</sup>,
    Mjaavatten O, **Vaudel M**, Bruserud Ø, Berven FS, Selheim F.
    *Reliable FASP-based procedures for optimal quantitative proteomic
    and phosphoproteomic analysis on samples from acute myeloid leukemia
    patients*, Biol Proced Online, 2016. \[[PubMed
    id 27330413](https://www.ncbi.nlm.nih.gov/pubmed/27330413)\]
    \[[doi: 10.1186/s12575-016-0043-0](https://doi.org/10.1186/s12575-016-0043-0)\]
    \[[pdf](https://biologicalproceduresonline.biomedcentral.com/track/pdf/10.1186/s12575-016-0043-0)\]

32. Aasebø E<sup>1</sup>, Mjaavatten O, **Vaudel M**, Farag Y, Selheim
    F, Berven FS, Bruserud Ø, Hernandez-Valladares
    M<sup>1,:email:</sup>. *Freezing effects on the acute myeloid
    leukemia cell proteome and phosphoproteome revealed using optimal
    quantitative workflows*, Journal of Proteomics, 2016. \[[PubMed
    id 27107777](https://www.ncbi.nlm.nih.gov/pubmed/27107777)\]
    \[[doi: 10.1016/j.jprot.2016.03.049](https://doi.org/10.1016/j.jprot.2016.03.049)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

33. Opsahl JA, **Vaudel M**, Guldbrandsen A, Aasebø E, Van Pesch V,
    Franciotta D, Myhr KM, Barsnes H, Berle M, Torkildsen Ø, Kroksveen
    AC, Berven FS<sup>:email:</sup>. *Label free analysis of human
    cerebrospinal fluid addressing various normalization strategies and
    revealing protein groups affected by multiple sclerosis*,
    Proteomics, 2016. \[[PubMed
    id 26841090](https://www.ncbi.nlm.nih.gov/pubmed/26841090)\]
    \[[doi: 10.1002/pmic.201500284](https://doi.org/10.1002/pmic.201500284)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

34. **Vaudel M**, Burkhart JM, Zahedi RP, Oveland E, Berven FS, Sickmann
    A, Martens L<sup>:email:</sup>, Barsnes H. *PeptideShaker enables
    reanalysis of mass spectrometry-derived proteomics datasets*, Nature
    Biotechnology, 2015. \[[PubMed
    id 25574629](https://www.ncbi.nlm.nih.gov/pubmed/25574629)\]
    \[[doi: 10.1038/nbt.3109](https://doi.org/10.1038/nbt.3109)\] \[[pdf
    (accepted version)](docs/aam/peptideshaker_submitted.pdf)\]
    \[[Request a reprint](mailto:marc.vaudel@uib.no)\]

35. Verheggen K, Maddelein D, Hulstaert N, Martens L<sup>:email:</sup>,
    Barsnes H, **Vaudel M**. *Pladipus enables universal distributed
    computing in proteomics bioinformatics*, Journal of Proteome
    Research, 2015. \[[PubMed
    id 26510693](https://www.ncbi.nlm.nih.gov/pubmed/26510693)\]
    \[[doi: 10.1021/acs.jproteome.5b00850](https://doi.org/10.1021/acs.jproteome.5b00850)\]
    \[[pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jproteome.5b00850)\]

36. Barsnes H<sup>:email:</sup>, **Vaudel M**, Martens L. *JSparklines:
    Making tabular proteomics data come alive*, Proteomics, 2015.
    \[[PubMed
    id 25422159](https://www.ncbi.nlm.nih.gov/pubmed/25422159)\]
    \[[doi: 10.1002/pmic.201400356](https://doi.org/10.1002/pmic.201400356)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

37. Bjørlykke Y, Vethe H, **Vaudel M**, Barsnes H, Berven FS, Tjora E,
    Raeder H<sup>:email:</sup>. *Carboxyl-Ester Lipase Maturity-Onset
    Diabetes of the Young Disease Protein Biomarkers in
    Secretin-Stimulated Duodenal Juice*, Journal of Proteome Research,
    2015. \[[PubMed
    id 25369532](https://www.ncbi.nlm.nih.gov/pubmed/25369532)\]
    \[[doi: 10.1021/pr500750z](https://doi.org/10.1021/pr500750z)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

38. Muth T<sup>1</sup>, Weilnböck L<sup>1</sup>, Rapp E, Huber CG,
    Martens L, **Vaudel M**<sup>:email:</sup>, Barsnes H. *DeNovoGUI: an
    open source graphical user interface for de novo sequencing of
    tandem mass spectra*, Journal of Proteome Research, 2014. \[[PubMed
    id 24295440](https://www.ncbi.nlm.nih.gov/pubmed/24295440)\]
    \[[doi: 10.1021/pr4008078](https://doi.org/10.1021/pr4008078)\]
    \[[pdf](https://doi.org/10.1021/pr4008078)\]

39. Helgeland E, Breivik LE, **Vaudel M**, Svendsen OS, Garberg H,
    Nordrehaug JE, Berven FS, Jonassen AK<sup>:email:</sup>. *Exploring
    the human plasma proteome for humoral mediators of remote ischemic
    preconditioning - a word of caution*, PLoS One, 2014. \[[PubMed
    id 25333471](https://www.ncbi.nlm.nih.gov/pubmed/25333471)\]
    \[[doi: 10.1371/journal.pone.0109279](https://doi.org/10.1371/journal.pone.0109279)\]
    \[[pdf](https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0109279&type=printable)\]

40. Vethe H, Finne K, Skogstrand T, **Vaudel M**, Vikse BE, Hultström M,
    Placier S, Scherer A, Tenstad O, Marti HP<sup>:email:</sup>.
    *Distinct protein signature of hypertension-induced damage in the
    renal proteome of the two-kidney, one-clip rat model*, Journal of
    Hypertension, 2015. \[[PubMed
    id 25304470](https://www.ncbi.nlm.nih.gov/pubmed/25304470)\]
    \[[doi: 10.1097/HJH.0000000000000370](https://doi.org/10.1097/HJH.0000000000000370)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

41. Aasebø E, **Vaudel M**, Mjaavatten O, Gaudsdal G, Van der Burgh A,
    Gjertsen BT, Døskeland SO, Brusserud O, Berven FS, Selheim
    F<sup>:email:</sup>. *Performance of super-SILAC based quantitative
    proteomics for comparison of different acute myeloid leukemia (AML)
    cell lines*, Proteomics, 2014. \[[PubMed
    id 25044641](https://www.ncbi.nlm.nih.gov/pubmed/25044641)\]
    \[[doi: 10.1002/pmic.201300448](https://doi.org/10.1002/pmic.201300448)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

42. **Vaudel M**, Venne S, Berven FS, Zahedi RP, Martens L, Barsnes
    H<sup>:email:</sup>. *Shedding light on black boxes in protein
    identification*, Proteomics, 2014. \[[PubMed
    id 24678044](https://www.ncbi.nlm.nih.gov/pubmed/24678044)\]
    \[[doi: 10.1002/pmic.201300488](https://doi.org/10.1002/pmic.201300488)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/16924/5/Shedding%20light%20on%20black%20boxes%20in%20protein%20identification%20%28postprint%29.pdf)\]

43. Beck F, Geiger J, Gambaryan S, Veit J, **Vaudel M**, Nollau P,
    Kohlbacher O, Martens L, Walter U, Sickmann A<sup>:email:</sup>,
    Zahedi RP<sup>:email:</sup>. *Time-resolved characterization of
    cAMP/PKA-dependent signaling reveals that platelet inhibition is a
    concerted process involving multiple signaling pathways*, Blood,
    2014. \[[PubMed
    id 24324209](https://www.ncbi.nlm.nih.gov/pubmed/24324209)\]
    \[[doi: 10.1182/blood-2013-07-512384](https://doi.org/10.1182/blood-2013-07-512384)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

44. **Vaudel M**, Breiter D, Beck F, Rahnenführer J, Martens
    L<sup>:email:</sup>, Zahedi RP. *D-score: a search engine
    independent MD-score*, Proteomics, 2013. \[[PubMed
    id 23307401](https://www.ncbi.nlm.nih.gov/pubmed/23307401)\]
    \[[doi: 10.1002/pmic.201200408](https://doi.org/10.1002/pmic.201200408)\]
    \[[pdf (accepted
    version)](docs/aam/D-score%20revised%20version.pdf)\] \[[Request a
    reprint](mailto:marc.vaudel@uib.no)\]

45. Burkhart JM<sup>1</sup>, **Vaudel M**<sup>1</sup>, Gambaryan S,
    Radau S, Walter U, Martens L, Geiger J<sup>:email:</sup>, Sickmann
    A, and Zahedi RP<sup>:email:</sup>. *The first comprehensive and
    quantitative analysis of human platelet protein composition allows
    the comparative analysis of structural and functional pathways in
    human platelets*, Blood, 2012. \[[PubMed
    id 22869793](https://www.ncbi.nlm.nih.gov/pubmed/22869793)\]
    \[[doi: 10.1182/blood-2012-04-416594](https://doi.org/10.1182/blood-2012-04-416594)\]
    \[[pdf](https://ashpublications.org/blood/article-pdf/120/15/e73/1359688/zh804112000e73.pdf)\]

46. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Radau S, Zahedi
    RP, Martens L<sup>:email:</sup>, Sickmann A. *Integral
    Quantification Accuracy estimation for Reporter Ion based
    quantitative proteomics (iQuARI)*, Journal of Proteome Research,
    2012. \[[PubMed
    id 22874012](https://www.ncbi.nlm.nih.gov/pubmed/22874012)\]
    \[[doi: 10.1021/pr300247u](https://doi.org/10.1021/pr300247u)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

47. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Breiter D, Zahedi
    RP, Sickmann A<sup>:email:</sup>, Martens L. *A complex standard for
    protein identification, designed by evolution*, Journal of Proteome
    Research, 2012. \[[PubMed
    id 22489649](https://www.ncbi.nlm.nih.gov/pubmed/22489649)\]
    \[[doi: 10.1021/pr300055q](https://doi.org/10.1021/pr300055q)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

48. **Vaudel M**<sup>1</sup>, Barsnes H<sup>1,:email:</sup>, Berven FS,
    Sickmann A, Martens L. *SearchGUI: an open-source graphical user
    interface for simultaneous OMSSA and X\!Tandem searches*,
    Proteomics, 2011. \[[PubMed
    id 21337703](https://www.ncbi.nlm.nih.gov/pubmed/21337703)\]
    \[[doi: 10.1002/pmic.201000595](https://doi.org/10.1002/pmic.201000595)\]
    \[[pdf (accepted
    version)](docs/aam/Vaudel_et_al_SearchGUI_17092010.pdf)\] \[[Request
    a reprint](mailto:marc.vaudel@uib.no)\]

49. Colaert N, Barsnes H, **Vaudel M**, Helsens K, Timmerman E, Sickmann
    A, Gevaert K, Martens L<sup>:email:</sup>. *Thermo-msf-parser: an
    open source Java library to parse and visualize Thermo Proteome
    Discoverer msf files*, Journal of Proteome Research, 2011. \[[PubMed
    id 21714566](https://www.ncbi.nlm.nih.gov/pubmed/21714566)\]
    \[[doi: 10.1021/pr2005154](https://doi.org/10.1021/pr2005154)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/16679/3/Colaert%2bet%2bal%2b%2528postprint%2529.pdf)\]

50. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Sickmann A,
    Martens L<sup>:email:</sup>, Zahedi RP. *Peptide identification
    quality control*, Proteomics, 2011. \[[PubMed
    id 21500347](https://www.ncbi.nlm.nih.gov/pubmed/21500347)\]
    \[[doi: 10.1002/pmic.201000704](https://doi.org/10.1002/pmic.201000704)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

51. Barsnes H, **Vaudel M**, Colaert N, Helsens K, Sickmann A, Berven
    FS, Martens L<sup>:email:</sup>. *compomics-utilities: an
    open-source Java library for computational proteomics*, BMC
    Bioinformatics, 2011. \[[PubMed
    id 21385435](https://www.ncbi.nlm.nih.gov/pubmed/21385435)\]
    \[[doi: 10.1186/1471-2105-12-70](https://doi.org/10.1186/1471-2105-12-70)\]
    \[[pdf](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-70)\]

52. Burkhart JM<sup>1</sup>, **Vaudel M**<sup>1</sup>, Zahedi RP,
    Martens L, Sickmann A<sup>:email:</sup>. *iTRAQ protein
    quantification: a quality-controlled workflow*, Proteomics, 2011.
    \[[PubMed
    id 21328540](https://www.ncbi.nlm.nih.gov/pubmed/21328540)\]
    \[[doi: 10.1002/pmic.201000711](https://doi.org/10.1002/pmic.201000711)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

53. Muth T, **Vaudel M**, Barsnes H, Martens L<sup>:email:</sup>,
    Sickmann A. *XTandem Parser: An open-source library to parse and
    analyse X\!Tandem MS/MS search results*, Proteomics, 2010. \[[PubMed
    id 20140905](https://www.ncbi.nlm.nih.gov/pubmed/20140905)\]
    \[[doi: 10.1002/pmic.200900759](https://doi.org/10.1002/pmic.200900759)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## Outreach - Proceeding

1.  Burger B<sup>:email:</sup>, **Vaudel M**<sup>:email:</sup>, Barsnes
    H. *Importance of Block Randomization When Designing Proteomics
    Experiments*, Journal of Proteome Research, 2020. \[[PubMed
    id 32969222](https://www.ncbi.nlm.nih.gov/pubmed/%2032969222)\]
    \[[doi: 10.1021/acs.jproteome.0c00536](https://doi.org/10.1021/acs.jproteome.0c00536)\]
    \[[pdf](https://pubs.acs.org/doi/pdf/10.1021/acs.jproteome.0c00536)\]

2.  Njølstad PR<sup>:email:</sup>, Andreassen OA, Brunak S, Børglum AD,
    Dillner J, Esko T, Franks PW, Freimer N, Groop L, Heimer H, Hougaard
    DM, Hovig E, Hveem K, Jalanko A, Kaprio J, Knudsen GP, Melbye M,
    Metspalu A, Mortensen PB, Palmgren J, Palotie A, Reed W, Stefánsson
    H, Stitziel NO, Sullivan PF, Thorsteinsdóttir U, **Vaudel M**,
    Vuorio E, Werge T, Stoltenberg C, Stefánsson K. *Roadmap for a
    precision-medicine initiative in the Nordic region*, Nature
    Genetics, 2019. \[[PubMed
    id 30988515](https://www.ncbi.nlm.nih.gov/pubmed/30988515)\]
    \[[doi: 10.1038/s41588-019-0391-1](https://doi.org/10.1038/s41588-019-0391-1)\]
    \[[pdf](https://helda.helsinki.fi/bitstream/10138/325612/1/kaprio_48572_2_merged_1552149424_v2.pdf)\]

3.  Grüning BA, Lampa S, **Vaudel M**, Blankenberg D<sup>:email:</sup>.
    *Software engineering for scientific big data analysis*,
    Gigascience, 2019. \[[PubMed
    id 31121028](https://www.ncbi.nlm.nih.gov/pubmed/31121028)\]
    \[[doi: 10.1093/gigascience/giz054](https://doi.org/10.1093/gigascience/giz054)\]
    \[[pdf](https://academic.oup.com/gigascience/article-pdf/8/5/giz054/28698071/giz054.pdf)\]

4.  Willems S, bouyssié D, Deforce D, Dorfer V, Gorshkov V, Kopczynski
    D, Laukens K, Locard-Paulet M, Schwämml V, Uszkoreit J, Valkenborg
    D, **Vaudel M**, Bittremieux W<sup>:email:</sup>. *Proceedings of
    the EuBIC developer’s meeting 2018*, Journal of Proteomics, 2018.
    \[[PubMed
    id 29864591](https://www.ncbi.nlm.nih.gov/pubmed/29864591)\]
    \[[doi: 10.1016/j.jprot.2018.05.015](https://doi.org/10.1016/j.jprot.2018.05.015)\]
    \[[pdf](https://repository.uantwerpen.be/docman/irua/23a1a2/151764_2019_06_03.pdf)\]

5.  Willems S, Bouyssié D, David M, Locard-Paulet M, Mechtler K,
    Schwämmle V, Uszkoreit J, **Vaudel M**, Dorfer V<sup>:email:</sup>.
    *Proceedings of the EuBIC Winter School 2017*, Journal of
    Proteomics, 2017. \[[PubMed
    id 28385664](https://www.ncbi.nlm.nih.gov/pubmed/28385664)\]
    \[[doi: 10.1016/j.jprot.2017.04.001](https://doi.org/10.1016/j.jprot.2017.04.001)\]
    \[[pdf](https://air.unimi.it/bitstream/2434/803373/2/1-s2.0-S2212968519300078-main.pdf)\]

6.  **Vaudel M**, Veit Schwämmle<sup>:email:</sup>, Viktoria Dorfer.
    *The EuPA Bioinformatics Community (EuBIC) initiative*, EuPA Open
    Proteomics, 2016. \[[PubMed
    id](https://www.ncbi.nlm.nih.gov/pubmed/)\]
    \[[doi: 10.1016/j.euprot.2016.03.009](https://doi.org/10.1016/j.euprot.2016.03.009)\]
    \[[pdf](https://doi.org/10.1016/j.euprot.2016.03.009)\]

7.  Guldbrandsen A, **Vaudel M**, Barsnes H<sup>:email:</sup>.
    *Framtiden for biomedisinsk forskning - store endringer og nye
    muligheter*, Best Practice Nordic, 2016.
    [\[Link\]](https://bestprac.no/framtiden-biomedisinsk-forskning-store-endringer-gir-nye-muligheter/)

<br>

## Reviews

1.  Muth T<sup>:email:</sup>, Hartkopf F, **Vaudel
    M**<sup>:email:</sup>, Renard BY. *A Potential Golden Age to
    Come-Current Tools, Recent Use Cases, and Future Avenues for De Novo
    Sequencing in Proteomics*, Proteomics, 2018. \[[PubMed
    id 29968278](https://www.ncbi.nlm.nih.gov/pubmed/29968278)\]
    \[[doi: 10.1002/pmic.201700150](https://doi.org/10.1002/pmic.201700150)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/20984/3/De%2bnovo%2bsequencing%2breview_revised_final.pdf)\]

2.  Verheggen K, Ræder H, Breven FS, Martens L<sup>:email:</sup>,
    Barsnes H, **Vaudel M**. *Anatomy and evolution of database search
    engines—a central component of mass spectrometry based proteomic
    workflows*, Mass Spectrometry Reviews, 2017. \[[PubMed
    id 28902424](https://www.ncbi.nlm.nih.gov/pubmed/28902424)\]
    \[[doi: 10.1002/mas.21543](https://doi.org/10.1002/mas.21543)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/21005/4/Verheggen%2bet%2bal.%2bProteomic%2bdatabase%2bsearch%2bengines%2b-%2bfinal%2bversion.pdf)\]

3.  Hernandez-Valladares M<sup>:email:</sup>, **Vaudel M**, Selheim F,
    Berven F, Bruserud Ø. *Proteogenomics approaches for studying cancer
    biology and their potential in the identification of Acute Myeloid
    Leukemia biomarkers*, Expert Reviews of Proteomics, 2017. \[[PubMed
    id 28693350](https://www.ncbi.nlm.nih.gov/pubmed/28693350)\]
    \[[doi: 10.1080/14789450.2017.1352474](https://doi.org/10.1080/14789450.2017.1352474)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

4.  **Vaudel M**, Verheggen K, Csordas A, Raeder H, Berven FS, Martens
    L, Vizcaíno JA, Barsnes H<sup>:email:</sup>. *Exploring the
    potential of public proteomics data*, Proteomics, 2015. \[[PubMed
    id 26449181](https://www.ncbi.nlm.nih.gov/pubmed/26449181)\]
    \[[doi: 10.1002/pmic.201500295](https://doi.org/10.1002/pmic.201500295)\]
    \[[pdf](https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/pmic.201500295)\]

5.  **Vaudel M**<sup>:email:</sup>, Barsnes H, Bjerkvig R, Bikfalvi A,
    Selheim F, Berven FS, Daubon T. *Practical considerations for omics
    experiments in biomedical sciences*, Current Pharmaceutical
    Biotechnology, 2015. \[[PubMed
    id 26278526](https://www.ncbi.nlm.nih.gov/pubmed/26278526)\]
    \[[doi: 10.2174/1389201016666150817095348](https://doi.org/10.2174/1389201016666150817095348)\]
    \[[pdf](https://bora.uib.no/bora-xmlui/bitstream/1956/21252/4/review%2bcpb%2bleukemia%2b07.07.15.pdf)\]

6.  **Vaudel M**<sup>:email:</sup>, Sickmann A, Martens L. *Introduction
    to opportunities and pitfalls in functional mass spectrometry based
    proteomics*, Biochim Biophys Acta, 2014. \[[PubMed
    id 23845992](https://www.ncbi.nlm.nih.gov/pubmed/23845992)\]
    \[[doi: 10.1016/j.bbapap.2013.06.019](https://doi.org/10.1016/j.bbapap.2013.06.019)\]
    \[[pdf](https://biblio.ugent.be/publication/4259751/file/4286995.pdf)\]

7.  **Vaudel M**, Sickmann A, Martens L<sup>:email:</sup>. *Current
    methods for global proteome identification*, Expert Reviews of
    Proteomics, 2012. \[[PubMed
    id 23194269](https://www.ncbi.nlm.nih.gov/pubmed/23194269)\]
    \[[doi: 10.1586/epr.12.51](https://doi.org/10.1586/epr.12.51)\] \[No
    Open Access found [Request a reprint](mailto:marc.vaudel@uib.no)\]

8.  **Vaudel M**<sup>:email:</sup>, Sickmann A, Martens L. *Peptide and
    protein quantification: a map of the minefield*, Proteomics, 2010.
    \[[PubMed
    id 19953549](https://www.ncbi.nlm.nih.gov/pubmed/19953549)\]
    \[[doi: 10.1002/pmic.200900481](https://doi.org/10.1002/pmic.200900481)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## Book Chapters

1.  **Vaudel M**<sup>:email:</sup>. *Proteomics*, Processing
    Metabolomics and Proteomics Data with Open Software: A Practical
    Guide, 2019. \[PubMed\]
    \[[doi: 10.1039/9781788019880-00096](https://doi.org/10.1039/9781788019880-00096)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

2.  Schwämmle V<sup>:email:</sup>, **Vaudel M**. *Computational and
    Statistical Methods for High-Throughput Mass Spectrometry-Based PTM
    Analysis*, Methods in Molecular Biology: Protein Bioinformatics,
    Springer, 2017. \[[PubMed
    id 28150251](https://www.ncbi.nlm.nih.gov/pubmed/28150251)\]
    \[[doi: 10.1007/978-1-4939-6783-4\_21](https://doi.org/10.1007/978-1-4939-6783-4_21)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

3.  **Vaudel M**<sup>:email:</sup>. *MS2-Based Quantitation*, Proteome
    Informatics, Royal Society of Chemistry, 2016. \[PubMed\]
    \[[doi: 10.1039/9781782626732-00155](https://doi.org/10.1039/9781782626732-00155)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

4.  Lereim RR, Oveland E, Berven FS, **Vaudel M**<sup>:email:</sup>,
    Barsnes H. *Visualization, Inspection and Interpretation of Shotgun
    Proteomics Identification Results*, Advances in Experimental
    Medicine and Biology: Modern Proteomics – Sample Preparation,
    Analysis and Practical Applications, Springer, 2016. \[[PubMed
    id 27975220](https://www.ncbi.nlm.nih.gov/pubmed/27975220)\]
    \[[doi: 10.1007/978-3-319-41448-5\_11](https://doi.org/10.1007/978-3-319-41448-5_11)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

5.  Muth T, Rapp E, Berven FS, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Tandem Mass Spectrum Sequencing: An
    Alternative to Database Search Engines in Shotgun Proteomics*,
    Advances in Experimental Medicine and Biology: Modern Proteomics –
    Sample Preparation, Analysis and Practical Applications, Springer,
    2016. \[[PubMed
    id 27975219](https://www.ncbi.nlm.nih.gov/pubmed/27975219)\]
    \[[doi: 10.1007/978-3-319-41448-5\_10](https://doi.org/10.1007/978-3-319-41448-5_10)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

6.  Verheggen K, Martens L, Berven FS, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Database Search Engines: Paradigms,
    Challenges and Solutions*, Advances in Experimental Medicine and
    Biology: Modern Proteomics – Sample Preparation, Analysis and
    Practical Applications, Springer, 2016. \[[PubMed
    id 27975215](https://www.ncbi.nlm.nih.gov/pubmed/27975215)\]
    \[[doi: 10.1007/978-3-319-41448-5\_6](https://doi.org/10.1007/978-3-319-41448-5_6)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

7.  **Vaudel M**<sup>:email:</sup>, Barsnes H, Ræder H, Berven FS.
    *Using Proteomics Bioinformatics Tools and Resources in
    Proteogenomic Studies*, Advances in Experimental Medicine and
    Biology: Proteogenomics, Springer, 2016. \[[PubMed
    id 27686806](https://www.ncbi.nlm.nih.gov/pubmed/27686806)\]
    \[[doi: 10.1007/978-3-319-42316-6\_5](https://doi.org/10.1007/978-3-319-42316-6_5)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

8.  Hernández Sánchez LF, Aasebø E, Selheim F, Berven FS, Ræder H,
    Barsnes H, **Vaudel M**<sup>:email:</sup>. *Systemic Analysis of
    Regulated Functional Networks*, Methods in Molecular Biology:
    Proteomics in Systems Biology, Springer, 2015. \[[PubMed
    id 26700057](https://www.ncbi.nlm.nih.gov/pubmed/26700057)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_21](https://doi.org/10.1007/978-1-4939-3341-9_21)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

9.  Guldbrandsen A, Barsnes H, Kroksveen AC, Berven FS, **Vaudel
    M**<sup>:email:</sup>. *A Simple Workflow for Large Scale Shotgun
    Glycoproteomics*, Methods in Molecular Biology: Proteomics in
    Systems Biology, Springer, 2015. \[[PubMed
    id 26700056](https://www.ncbi.nlm.nih.gov/pubmed/26700056)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_20](https://doi.org/10.1007/978-1-4939-3341-9_20)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

10. Aasebø E, Berven FS, Selheim F, Barsnes H, **Vaudel
    M**<sup>:email:</sup>. *Interpretation of Quantitative Shotgun
    Proteomic Data*, Methods in Molecular Biology: Proteomics in Systems
    Biology, Springer, 2015. \[[PubMed
    id 26700055](https://www.ncbi.nlm.nih.gov/pubmed/26700055)\]
    \[[doi: 10.1007/978-1-4939-3341-9\_19](https://doi.org/10.1007/978-1-4939-3341-9_19)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

11. **Vaudel M**<sup>:email:</sup>, Barsnes H, Martens L, Berven FS.
    *Bioinformatics for Proteomics: Opportunities at the interface
    between the scientists, their experiments and the community*,
    Methods in Molecular Biology: Shotgun Proteomics, Springer, 2015.
    \[[PubMed
    id 24791993](https://www.ncbi.nlm.nih.gov/pubmed/24791993)\]
    \[[doi: 10.1007/978-1-4939-0685-7\_16](https://doi.org/10.1007/978-1-4939-0685-7_16)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

12. **Vaudel M**<sup>1</sup>, Burkhart JM<sup>1</sup>, Zahedi RP,
    Martens L, Sickmann A<sup>:email:</sup>. *iTRAQ data
    interpretation*, Methods in Molecular Biology: Quantitative Methods
    in Proteomics, Springer, 2012. \[[PubMed
    id 22665319](https://www.ncbi.nlm.nih.gov/pubmed/22665319)\]
    \[[doi: 10.1007/978-1-61779-885-6\_30](https://doi.org/10.1007/978-1-61779-885-6_30)\]
    \[No Open Access found [Request a
    reprint](mailto:marc.vaudel@uib.no)\]

<br>

## PhD thesis

*Detailed analysis of the blood platelet proteome through the use of
innovative bioinformatics methods for protein identification,
quantification and post-translational modification localization,* Ghent
University, Faculty of Medicine and Health Sciences, 2012 (monograph).
[\[link\]](https://hdl.handle.net/1854/LU-3076565)

<br>

## About

This document was generated from this [Rmd](README.Rmd). Links to pdfs
are obtained from [Unpaywall](unpaywall.org), queried using the
[roadoi](https://cran.r-project.org/web/packages/roadoi/vignettes/intro.html)
package.

For questions or error report, please do not hesitate to reach out *via*
the [issue tracker](https://github.com/mvaudel/Publications/issues).

For personal use only, no copyright infringement intended. <br>

## R Session Info

``` r
sessionInfo()
```

    ## R version 3.6.1 (2019-07-05)
    ## Platform: x86_64-w64-mingw32/x64 (64-bit)
    ## Running under: Windows 10 x64 (build 17134)
    ## 
    ## Matrix products: default
    ## 
    ## locale:
    ## [1] LC_COLLATE=Norwegian Bokmål_Norway.1252 
    ## [2] LC_CTYPE=Norwegian Bokmål_Norway.1252   
    ## [3] LC_MONETARY=Norwegian Bokmål_Norway.1252
    ## [4] LC_NUMERIC=C                            
    ## [5] LC_TIME=Norwegian Bokmål_Norway.1252    
    ## 
    ## attached base packages:
    ## [1] stats     graphics  grDevices utils     datasets  methods   base     
    ## 
    ## other attached packages:
    ## [1] roadoi_0.6       conflicted_1.0.4
    ## 
    ## loaded via a namespace (and not attached):
    ##  [1] Rcpp_1.0.4.6     pillar_1.4.6     compiler_3.6.1   later_1.1.0.1   
    ##  [5] plyr_1.8.6       tools_3.6.1      digest_0.6.25    jsonlite_1.7.0  
    ##  [9] evaluate_0.14    lifecycle_1.0.1  tibble_3.0.0     pkgconfig_2.0.3 
    ## [13] rlang_0.4.10     shiny_1.5.0      cli_3.0.1        rstudioapi_0.13 
    ## [17] curl_4.3         yaml_2.2.1       xfun_0.16        fastmap_1.0.1   
    ## [21] dplyr_1.0.0      stringr_1.4.0    httr_1.4.2       knitr_1.29      
    ## [25] generics_0.0.2   vctrs_0.3.8      tidyselect_1.1.0 glue_1.4.0      
    ## [29] R6_2.5.1         rmarkdown_2.3    purrr_0.3.4      tidyr_1.1.3     
    ## [33] magrittr_1.5     promises_1.1.1   ellipsis_0.3.0   htmltools_0.5.0 
    ## [37] mime_0.9         xtable_1.8-4     httpuv_1.5.4     stringi_1.4.6   
    ## [41] miniUI_0.1.1.1   cachem_1.0.5     crayon_1.3.4
